{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%matplotlib inline\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "from IPython.core.pylabtools import figsize\n",
    "import matplotlib.pyplot as plt\n",
    "import os.path\n",
    "import pandas as pd\n",
    "import collections\n",
    "import seaborn as sns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>contig</th>\n",
       "      <th>resolved_locus</th>\n",
       "      <th>stacks_CatID</th>\n",
       "      <th>stacks_SNP</th>\n",
       "      <th>LEPname</th>\n",
       "      <th>LEP_LG</th>\n",
       "      <th>cM</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>6988</th>\n",
       "      <td>c2953</td>\n",
       "      <td>2953_C</td>\n",
       "      <td>16919</td>\n",
       "      <td>16919_25</td>\n",
       "      <td>2902</td>\n",
       "      <td>37</td>\n",
       "      <td>56.28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6989</th>\n",
       "      <td>c56816</td>\n",
       "      <td>56816_C</td>\n",
       "      <td>39923</td>\n",
       "      <td>39923_44</td>\n",
       "      <td>7366</td>\n",
       "      <td>37</td>\n",
       "      <td>56.28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6990</th>\n",
       "      <td>c47181</td>\n",
       "      <td>47181_C</td>\n",
       "      <td>31760</td>\n",
       "      <td>31760_82</td>\n",
       "      <td>5379</td>\n",
       "      <td>37</td>\n",
       "      <td>56.28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6991</th>\n",
       "      <td>c7090</td>\n",
       "      <td>7090_A</td>\n",
       "      <td>51610</td>\n",
       "      <td>51610_40</td>\n",
       "      <td>10169</td>\n",
       "      <td>37</td>\n",
       "      <td>56.28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6992</th>\n",
       "      <td>c56688</td>\n",
       "      <td>56688_C</td>\n",
       "      <td>39817</td>\n",
       "      <td>39817_67</td>\n",
       "      <td>7337</td>\n",
       "      <td>37</td>\n",
       "      <td>66.16</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      contig resolved_locus  stacks_CatID stacks_SNP  LEPname  LEP_LG     cM\n",
       "6988   c2953         2953_C         16919   16919_25     2902      37  56.28\n",
       "6989  c56816        56816_C         39923   39923_44     7366      37  56.28\n",
       "6990  c47181        47181_C         31760   31760_82     5379      37  56.28\n",
       "6991   c7090         7090_A         51610   51610_40    10169      37  56.28\n",
       "6992  c56688        56688_C         39817   39817_67     7337      37  66.16"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lmap = pd.read_csv(os.path.join('linkage_map','LEPmap','with_paralogs','final','PS_chum_map_2015.txt'), sep = '\\t')\n",
    "lmap = lmap.drop(['cM_OLD', 'paper1_LG'], axis = 1)\n",
    "lmap= lmap.drop_duplicates('LEPname')\n",
    "lmap = lmap.sort_values(['LEP_LG', 'cM'])\n",
    "lmap.tail()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "lmap['fakeBP'] = (lmap['cM']*1000000 + lmap.index).astype(np.int)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>contig</th>\n",
       "      <th>resolved_locus</th>\n",
       "      <th>stacks_CatID</th>\n",
       "      <th>stacks_SNP</th>\n",
       "      <th>LEPname</th>\n",
       "      <th>LEP_LG</th>\n",
       "      <th>cM</th>\n",
       "      <th>fakeBP</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>6988</th>\n",
       "      <td>c2953</td>\n",
       "      <td>2953_C</td>\n",
       "      <td>16919</td>\n",
       "      <td>16919_25</td>\n",
       "      <td>2902</td>\n",
       "      <td>37</td>\n",
       "      <td>56.28</td>\n",
       "      <td>56286988</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6989</th>\n",
       "      <td>c56816</td>\n",
       "      <td>56816_C</td>\n",
       "      <td>39923</td>\n",
       "      <td>39923_44</td>\n",
       "      <td>7366</td>\n",
       "      <td>37</td>\n",
       "      <td>56.28</td>\n",
       "      <td>56286989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6990</th>\n",
       "      <td>c47181</td>\n",
       "      <td>47181_C</td>\n",
       "      <td>31760</td>\n",
       "      <td>31760_82</td>\n",
       "      <td>5379</td>\n",
       "      <td>37</td>\n",
       "      <td>56.28</td>\n",
       "      <td>56286990</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6991</th>\n",
       "      <td>c7090</td>\n",
       "      <td>7090_A</td>\n",
       "      <td>51610</td>\n",
       "      <td>51610_40</td>\n",
       "      <td>10169</td>\n",
       "      <td>37</td>\n",
       "      <td>56.28</td>\n",
       "      <td>56286991</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6992</th>\n",
       "      <td>c56688</td>\n",
       "      <td>56688_C</td>\n",
       "      <td>39817</td>\n",
       "      <td>39817_67</td>\n",
       "      <td>7337</td>\n",
       "      <td>37</td>\n",
       "      <td>66.16</td>\n",
       "      <td>66166992</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      contig resolved_locus  stacks_CatID stacks_SNP  LEPname  LEP_LG     cM  \\\n",
       "6988   c2953         2953_C         16919   16919_25     2902      37  56.28   \n",
       "6989  c56816        56816_C         39923   39923_44     7366      37  56.28   \n",
       "6990  c47181        47181_C         31760   31760_82     5379      37  56.28   \n",
       "6991   c7090         7090_A         51610   51610_40    10169      37  56.28   \n",
       "6992  c56688        56688_C         39817   39817_67     7337      37  66.16   \n",
       "\n",
       "        fakeBP  \n",
       "6988  56286988  \n",
       "6989  56286989  \n",
       "6990  56286990  \n",
       "6991  56286991  \n",
       "6992  66166992  "
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lmap.tail()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "mapfile = lmap[['LEP_LG', 'LEPname', 'cM', 'fakeBP']]\n",
    "mapfile.to_csv('/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.map', \n",
    "               sep = '\\t', index = False, header = False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>LEP_LG</th>\n",
       "      <th>fakeBP</th>\n",
       "      <th>rrate</th>\n",
       "      <th>cM</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1</td>\n",
       "      <td>2</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>1</td>\n",
       "      <td>6</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>1</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>1</td>\n",
       "      <td>14</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    LEP_LG  fakeBP  rrate  cM\n",
       "0        1       0      1   0\n",
       "2        1       2      1   0\n",
       "6        1       6      1   0\n",
       "12       1      12      1   0\n",
       "14       1      14      1   0"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "shapeit_mapfile = lmap[['LEP_LG','fakeBP', 'cM']].copy()\n",
    "shapeit_mapfile['rrate'] = 1.0\n",
    "shapeit_mapfile = shapeit_mapfile[['LEP_LG', 'fakeBP', 'rrate', 'cM']].copy()\n",
    "shapeit_mapfile.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "gb = shapeit_mapfile.groupby('LEP_LG')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "for xx in range(1, 38):\n",
    "    to_write = gb.get_group(xx).drop('LEP_LG', axis = 1)\n",
    "    to_write.to_csv('linkage_map/centromere_placement/shapeit/collapsed_loci.{}.shapeitmap'.format(xx), sep = ' ', header = True, index = False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#family</th>\n",
       "      <th>name</th>\n",
       "      <th>sire</th>\n",
       "      <th>dam</th>\n",
       "      <th>sex</th>\n",
       "      <th>blank</th>\n",
       "      <th>10001</th>\n",
       "      <th>10003_A</th>\n",
       "      <th>1000_C</th>\n",
       "      <th>10012</th>\n",
       "      <th>...</th>\n",
       "      <th>Oke_ndub3-58</th>\n",
       "      <th>Oke_pnrc2-78</th>\n",
       "      <th>Oke_psmd9-57</th>\n",
       "      <th>Oke_rab5a-117</th>\n",
       "      <th>Oke_sylc-90</th>\n",
       "      <th>Oke_thic-84</th>\n",
       "      <th>Oke_txnrd1-74</th>\n",
       "      <th>Oke_u0602-244</th>\n",
       "      <th>Oke_u217-172</th>\n",
       "      <th>Oke_zn593-152</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>fam_08_Dam</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>2</td>\n",
       "      <td>0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>...</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>fam_08_Sire</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>...</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>CMUW10X08H_0002</td>\n",
       "      <td>fam_08_Sire</td>\n",
       "      <td>fam_08_Dam</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>...</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>CMUW10X08H_0003</td>\n",
       "      <td>fam_08_Sire</td>\n",
       "      <td>fam_08_Dam</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 1</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>...</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>CMUW10X08H_0005</td>\n",
       "      <td>fam_08_Sire</td>\n",
       "      <td>fam_08_Dam</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>...</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 12157 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "  #family             name         sire         dam  sex  blank 10001 10003_A  \\\n",
       "0  fam_08       fam_08_Dam            0           0    2      0   1 1     1 1   \n",
       "1  fam_08      fam_08_Sire            0           0    1      0   1 2     1 2   \n",
       "2  fam_08  CMUW10X08H_0002  fam_08_Sire  fam_08_Dam    0      0   1 1     1 2   \n",
       "3  fam_08  CMUW10X08H_0003  fam_08_Sire  fam_08_Dam    0      0   1 2     1 1   \n",
       "4  fam_08  CMUW10X08H_0005  fam_08_Sire  fam_08_Dam    0      0   1 2     1 2   \n",
       "\n",
       "  1000_C 10012      ...      Oke_ndub3-58 Oke_pnrc2-78 Oke_psmd9-57  \\\n",
       "0    1 1   1 1      ...               1 1          1 1          1 1   \n",
       "1    1 2   1 2      ...               1 2          1 2          1 2   \n",
       "2    0 0   0 0      ...               0 0          1 2          0 0   \n",
       "3    0 0   0 0      ...               0 0          1 2          0 0   \n",
       "4    0 0   0 0      ...               0 0          1 2          0 0   \n",
       "\n",
       "  Oke_rab5a-117 Oke_sylc-90 Oke_thic-84 Oke_txnrd1-74 Oke_u0602-244  \\\n",
       "0           1 1         1 1         1 1           1 1           1 1   \n",
       "1           1 2         1 2         1 2           1 2           1 2   \n",
       "2           0 0         1 1         1 2           1 2           1 1   \n",
       "3           0 0         1 1         1 1           1 2           1 2   \n",
       "4           0 0         1 1         1 2           1 2           1 2   \n",
       "\n",
       "  Oke_u217-172 Oke_zn593-152  \n",
       "0          1 1           1 1  \n",
       "1          1 2           1 2  \n",
       "2          1 1           1 2  \n",
       "3          1 1           1 1  \n",
       "4          1 2           1 2  \n",
       "\n",
       "[5 rows x 12157 columns]"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lepmap_infile = pd.read_csv(os.path.join('linkage_map','LEPmap','with_paralogs', 'collapsed_loci.lepmap'), sep = '\\t')\n",
    "lepmap_infile.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "lepmap_infile.columns = lepmap_infile.columns[:6].tolist() + list(range(1,len(lepmap_infile.columns[6:])+1))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## subset and reorder base on linkage map"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#family</th>\n",
       "      <th>name</th>\n",
       "      <th>sire</th>\n",
       "      <th>dam</th>\n",
       "      <th>sex</th>\n",
       "      <th>blank</th>\n",
       "      <th>4584</th>\n",
       "      <th>7377</th>\n",
       "      <th>6722</th>\n",
       "      <th>1923</th>\n",
       "      <th>...</th>\n",
       "      <th>4552</th>\n",
       "      <th>6213</th>\n",
       "      <th>2558</th>\n",
       "      <th>5198</th>\n",
       "      <th>5942</th>\n",
       "      <th>2902</th>\n",
       "      <th>7366</th>\n",
       "      <th>5379</th>\n",
       "      <th>10169</th>\n",
       "      <th>7337</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>fam_08_Dam</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>2</td>\n",
       "      <td>0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>...</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>fam_08_Sire</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>...</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>CMUW10X08H_0002</td>\n",
       "      <td>fam_08_Sire</td>\n",
       "      <td>fam_08_Dam</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>0 0</td>\n",
       "      <td>...</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>0 0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>CMUW10X08H_0003</td>\n",
       "      <td>fam_08_Sire</td>\n",
       "      <td>fam_08_Dam</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>0 0</td>\n",
       "      <td>...</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 1</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>CMUW10X08H_0005</td>\n",
       "      <td>fam_08_Sire</td>\n",
       "      <td>fam_08_Dam</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>0 0</td>\n",
       "      <td>...</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>0 0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 7801 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "  #family             name         sire         dam  sex  blank 4584 7377  \\\n",
       "0  fam_08       fam_08_Dam            0           0    2      0  1 1  1 1   \n",
       "1  fam_08      fam_08_Sire            0           0    1      0  1 2  1 2   \n",
       "2  fam_08  CMUW10X08H_0002  fam_08_Sire  fam_08_Dam    0      0  1 1  1 1   \n",
       "3  fam_08  CMUW10X08H_0003  fam_08_Sire  fam_08_Dam    0      0  1 1  1 1   \n",
       "4  fam_08  CMUW10X08H_0005  fam_08_Sire  fam_08_Dam    0      0  1 1  1 1   \n",
       "\n",
       "  6722 1923 ...  4552 6213 2558 5198 5942 2902 7366 5379 10169 7337  \n",
       "0  1 1  1 1 ...   1 1  1 1  1 1  1 1  1 1  1 1  1 1  1 1   1 1  1 1  \n",
       "1  1 2  1 2 ...   1 2  1 2  1 2  1 2  1 2  1 2  1 2  1 2   1 2  1 2  \n",
       "2  1 1  0 0 ...   1 1  1 2  0 0  1 1  1 2  0 0  0 0  0 0   1 1  0 0  \n",
       "3  1 1  0 0 ...   1 2  1 1  0 0  0 0  1 1  0 0  0 0  0 0   1 2  0 0  \n",
       "4  1 1  0 0 ...   1 1  1 2  0 0  1 1  1 2  0 0  0 0  0 0   1 1  0 0  \n",
       "\n",
       "[5 rows x 7801 columns]"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ped = lepmap_infile[lepmap_infile.columns[:6].tolist() + lmap['LEPname'].tolist()]\n",
    "ped.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## drop the two small families"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "ped = ped[ped['#family']=='fam_08']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "ped.to_csv('/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.ped', \n",
    "               sep = '\\t', index = False, header = False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Split by chromosome in PLINK"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 1\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (331 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1-temporary.fam\n",
      "written.\n",
      "331 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.605254.\n",
      "119 variants removed due to missing genotype data (--geno).\n",
      "212 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 2\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (300 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2-temporary.fam\n",
      "written.\n",
      "300 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.650414.\n",
      "95 variants removed due to missing genotype data (--geno).\n",
      "205 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 3\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (320 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3-temporary.fam\n",
      "written.\n",
      "320 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.596186.\n",
      "120 variants removed due to missing genotype data (--geno).\n",
      "200 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 4\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (270 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4-temporary.fam\n",
      "written.\n",
      "270 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.581042.\n",
      "104 variants removed due to missing genotype data (--geno).\n",
      "166 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 5\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (369 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5-temporary.fam\n",
      "written.\n",
      "369 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.450171.\n",
      "195 variants removed due to missing genotype data (--geno).\n",
      "174 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 6\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (320 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6-temporary.fam\n",
      "written.\n",
      "320 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.488224.\n",
      "157 variants removed due to missing genotype data (--geno).\n",
      "163 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 7\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (268 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7-temporary.fam\n",
      "written.\n",
      "268 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.534046.\n",
      "117 variants removed due to missing genotype data (--geno).\n",
      "151 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 8\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (282 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8-temporary.fam\n",
      "written.\n",
      "282 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.537825.\n",
      "123 variants removed due to missing genotype data (--geno).\n",
      "159 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 9\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (286 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9-temporary.fam\n",
      "written.\n",
      "286 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.526886.\n",
      "127 variants removed due to missing genotype data (--geno).\n",
      "159 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 10\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (274 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10-temporary.fam\n",
      "written.\n",
      "274 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.533836.\n",
      "121 variants removed due to missing genotype data (--geno).\n",
      "153 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 11\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (303 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11-temporary.fam\n",
      "written.\n",
      "303 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.482202.\n",
      "151 variants removed due to missing genotype data (--geno).\n",
      "152 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 12\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (260 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12-temporary.fam\n",
      "written.\n",
      "260 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.522642.\n",
      "117 variants removed due to missing genotype data (--geno).\n",
      "143 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 13\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (214 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13-temporary.fam\n",
      "written.\n",
      "214 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.456756.\n",
      "111 variants removed due to missing genotype data (--geno).\n",
      "103 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 14\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (207 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14-temporary.fam\n",
      "written.\n",
      "207 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.542891.\n",
      "89 variants removed due to missing genotype data (--geno).\n",
      "118 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 15\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (198 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15-temporary.fam\n",
      "written.\n",
      "198 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.563973.\n",
      "80 variants removed due to missing genotype data (--geno).\n",
      "118 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 16\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (184 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16-temporary.fam\n",
      "written.\n",
      "184 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.524472.\n",
      "82 variants removed due to missing genotype data (--geno).\n",
      "102 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 17\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (164 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17-temporary.fam\n",
      "written.\n",
      "164 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.57727.\n",
      "64 variants removed due to missing genotype data (--geno).\n",
      "100 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 18\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (182 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18-temporary.fam\n",
      "written.\n",
      "182 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.578879.\n",
      "71 variants removed due to missing genotype data (--geno).\n",
      "111 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 19\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (206 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19-temporary.fam\n",
      "written.\n",
      "206 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.550518.\n",
      "87 variants removed due to missing genotype data (--geno).\n",
      "119 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 20\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (175 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20-temporary.fam\n",
      "written.\n",
      "175 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.504859.\n",
      "82 variants removed due to missing genotype data (--geno).\n",
      "93 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 21\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (175 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21-temporary.fam\n",
      "written.\n",
      "175 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.548249.\n",
      "74 variants removed due to missing genotype data (--geno).\n",
      "101 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 22\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (185 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22-temporary.fam\n",
      "written.\n",
      "185 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.517667.\n",
      "84 variants removed due to missing genotype data (--geno).\n",
      "101 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 23\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (196 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23-temporary.fam\n",
      "written.\n",
      "196 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.542575.\n",
      "85 variants removed due to missing genotype data (--geno).\n",
      "111 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 24\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (156 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24-temporary.fam\n",
      "written.\n",
      "156 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.606982.\n",
      "56 variants removed due to missing genotype data (--geno).\n",
      "100 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 25\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (167 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25-temporary.fam\n",
      "written.\n",
      "167 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.466017.\n",
      "85 variants removed due to missing genotype data (--geno).\n",
      "82 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 26\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (219 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26-temporary.fam\n",
      "written.\n",
      "219 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.490726.\n",
      "106 variants removed due to missing genotype data (--geno).\n",
      "113 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 27\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (153 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27-temporary.fam\n",
      "written.\n",
      "153 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.562424.\n",
      "63 variants removed due to missing genotype data (--geno).\n",
      "90 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 28\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (174 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28-temporary.fam\n",
      "written.\n",
      "174 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.592246.\n",
      "66 variants removed due to missing genotype data (--geno).\n",
      "108 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 29\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (151 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29-temporary.fam\n",
      "written.\n",
      "151 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.581509.\n",
      "59 variants removed due to missing genotype data (--geno).\n",
      "92 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 30\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (137 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30-temporary.fam\n",
      "written.\n",
      "137 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.570786.\n",
      "54 variants removed due to missing genotype data (--geno).\n",
      "83 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 31\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (162 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31-temporary.fam\n",
      "written.\n",
      "162 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.498466.\n",
      "77 variants removed due to missing genotype data (--geno).\n",
      "85 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 32\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (161 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32-temporary.fam\n",
      "written.\n",
      "161 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.530407.\n",
      "71 variants removed due to missing genotype data (--geno).\n",
      "90 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 33\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (125 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33-temporary.fam\n",
      "written.\n",
      "125 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.604249.\n",
      "45 variants removed due to missing genotype data (--geno).\n",
      "80 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 34\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (153 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34-temporary.fam\n",
      "written.\n",
      "153 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.460987.\n",
      "79 variants removed due to missing genotype data (--geno).\n",
      "74 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 35\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (124 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35-temporary.fam\n",
      "written.\n",
      "124 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.568161.\n",
      "50 variants removed due to missing genotype data (--geno).\n",
      "74 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 36\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (146 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36-temporary.fam\n",
      "written.\n",
      "146 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.398382.\n",
      "85 variants removed due to missing genotype data (--geno).\n",
      "61 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 37\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --geno .5\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (98 variants, 177 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37-temporary.fam\n",
      "written.\n",
      "98 variants loaded from .bim file.\n",
      "177 samples (1 male, 1 female, 175 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 2 founders and 175 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.621181.\n",
      "34 variants removed due to missing genotype data (--geno).\n",
      "64 variants and 177 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n"
     ]
    }
   ],
   "source": [
    "for chrom in range(1, 38):\n",
    "    !plink --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci \\\n",
    "           --autosome-num 50 --chr {chrom} --recode --geno .5 \\\n",
    "           --out {'/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom{}'.format(chrom)}"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Run SHAPEIT2 to phase"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:44:05\n",
      "  * LOGfile : [shapeit_30102015_23h44m05s_a88d02ce-7534-4184-a5d6-9a7596c9688b.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446273845\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.map]\n",
      "  * 212 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.1.shapeitmap]\n",
      "  * 331 genetic positions found\n",
      "  * #set=212 / #interpolated=0\n",
      "  * Physical map [0.00 Mb -> 106.08 Mb] / Genetic map [0.00 cM -> 106.08 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 82 individuals with high rates of missing data (>5%)\n",
      "  * 103 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 6855 segments / ~5 SNPs per segment / 348265 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 6344 segments / ~5 SNPs per segment / 185902 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 12 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:44:17\n",
      "  * LOGfile : [shapeit_30102015_23h44m17s_23aaeae9-acac-483b-a651-a9c2ef7b200c.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446273857\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.map]\n",
      "  * 205 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.2.shapeitmap]\n",
      "  * 300 genetic positions found\n",
      "  * #set=205 / #interpolated=0\n",
      "  * Physical map [0.00 Mb -> 111.35 Mb] / Genetic map [0.00 cM -> 111.35 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 69 individuals with high rates of missing data (>5%)\n",
      "  * 86 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 6478 segments / ~5 SNPs per segment / 312931 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 5912 segments / ~6 SNPs per segment / 176245 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 12 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:44:29\n",
      "  * LOGfile : [shapeit_30102015_23h44m29s_c4915283-14c5-4cfa-806e-389ca2193c94.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446273869\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.map]\n",
      "  * 200 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.3.shapeitmap]\n",
      "  * 320 genetic positions found\n",
      "  * #set=200 / #interpolated=0\n",
      "  * Physical map [2.65 Mb -> 129.59 Mb] / Genetic map [2.65 cM -> 129.59 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 65 individuals with high rates of missing data (>5%)\n",
      "  * 87 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 6352 segments / ~5 SNPs per segment / 304801 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 5741 segments / ~6 SNPs per segment / 170062 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 11 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:44:40\n",
      "  * LOGfile : [shapeit_30102015_23h44m40s_40e981e3-16cd-4137-986e-7b355fda2521.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446273880\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.map]\n",
      "  * 166 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.4.shapeitmap]\n",
      "  * 270 genetic positions found\n",
      "  * #set=166 / #interpolated=0\n",
      "  * Physical map [0.00 Mb -> 119.48 Mb] / Genetic map [0.00 cM -> 119.48 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 87 individuals with high rates of missing data (>5%)\n",
      "  * 85 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 5363 segments / ~5 SNPs per segment / 269904 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 4894 segments / ~6 SNPs per segment / 140443 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 11 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:44:51\n",
      "  * LOGfile : [shapeit_30102015_23h44m51s_da8bdfd8-6dbd-479a-a23f-5b731e5f659a.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446273891\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.map]\n",
      "  * 174 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.5.shapeitmap]\n",
      "  * 369 genetic positions found\n",
      "  * #set=174 / #interpolated=0\n",
      "  * Physical map [0.00 Mb -> 137.46 Mb] / Genetic map [0.00 cM -> 137.46 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 83 individuals with high rates of missing data (>5%)\n",
      "  * 80 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 5722 segments / ~5 SNPs per segment / 277973 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 5248 segments / ~5 SNPs per segment / 157205 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 11 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:45:02\n",
      "  * LOGfile : [shapeit_30102015_23h45m02s_63ef8334-e80d-4a08-babd-ee7f1c3acc2c.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446273902\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.map]\n",
      "  * 163 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.6.shapeitmap]\n",
      "  * 320 genetic positions found\n",
      "  * #set=163 / #interpolated=0\n",
      "  * Physical map [0.00 Mb -> 118.19 Mb] / Genetic map [0.00 cM -> 118.19 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 67 individuals with high rates of missing data (>5%)\n",
      "  * 62 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 5183 segments / ~5 SNPs per segment / 248046 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 4738 segments / ~6 SNPs per segment / 139694 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 11 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:45:13\n",
      "  * LOGfile : [shapeit_30102015_23h45m13s_60fe8484-f845-417b-b583-3b24c733d044.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446273913\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.map]\n",
      "  * 151 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.7.shapeitmap]\n",
      "  * 268 genetic positions found\n",
      "  * #set=151 / #interpolated=0\n",
      "  * Physical map [3.81 Mb -> 108.91 Mb] / Genetic map [3.80 cM -> 108.90 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 74 individuals with high rates of missing data (>5%)\n",
      "  * 81 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 4896 segments / ~5 SNPs per segment / 243566 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 4529 segments / ~5 SNPs per segment / 131241 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 10 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:45:23\n",
      "  * LOGfile : [shapeit_30102015_23h45m23s_c63ab0a8-fbb8-4e37-a6e0-1a8e7d9ecbca.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446273923\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.map]\n",
      "  * 159 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.8.shapeitmap]\n",
      "  * 282 genetic positions found\n",
      "  * #set=159 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 115.09 Mb] / Genetic map [0.00 cM -> 115.08 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 72 individuals with high rates of missing data (>5%)\n",
      "  * 75 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 5096 segments / ~5 SNPs per segment / 246579 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 4732 segments / ~5 SNPs per segment / 139456 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 10 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:45:33\n",
      "  * LOGfile : [shapeit_30102015_23h45m33s_8720f902-f8dc-410e-80ca-7f0bfc844c05.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446273933\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.map]\n",
      "  * 159 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.9.shapeitmap]\n",
      "  * 286 genetic positions found\n",
      "  * #set=159 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 113.91 Mb] / Genetic map [0.00 cM -> 113.90 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 86 individuals with high rates of missing data (>5%)\n",
      "  * 73 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 5169 segments / ~5 SNPs per segment / 253938 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 4761 segments / ~5 SNPs per segment / 138952 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 11 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:45:44\n",
      "  * LOGfile : [shapeit_30102015_23h45m44s_6e9724aa-c9a0-4d18-aa58-f2b3f3f9d3d1.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446273944\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.map]\n",
      "  * 153 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.10.shapeitmap]\n",
      "  * 274 genetic positions found\n",
      "  * #set=153 / #interpolated=0\n",
      "  * Physical map [10.46 Mb -> 127.00 Mb] / Genetic map [10.46 cM -> 127.00 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 69 individuals with high rates of missing data (>5%)\n",
      "  * 64 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 4860 segments / ~5 SNPs per segment / 235280 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 4370 segments / ~6 SNPs per segment / 126670 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 9 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:45:53\n",
      "  * LOGfile : [shapeit_30102015_23h45m53s_344b6386-d759-4784-98d9-b68f13804951.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446273953\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.map]\n",
      "  * 152 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.11.shapeitmap]\n",
      "  * 303 genetic positions found\n",
      "  * #set=152 / #interpolated=0\n",
      "  * Physical map [14.05 Mb -> 133.25 Mb] / Genetic map [14.05 cM -> 133.25 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 71 individuals with high rates of missing data (>5%)\n",
      "  * 59 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 4765 segments / ~5 SNPs per segment / 225596 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 4274 segments / ~6 SNPs per segment / 128976 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 9 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:46:03\n",
      "  * LOGfile : [shapeit_30102015_23h46m03s_d1994d15-4a09-4884-9935-6ab4bd3c5fb3.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446273963\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.map]\n",
      "  * 143 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.12.shapeitmap]\n",
      "  * 260 genetic positions found\n",
      "  * #set=143 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 127.12 Mb] / Genetic map [0.00 cM -> 127.11 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 75 individuals with high rates of missing data (>5%)\n",
      "  * 70 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 4594 segments / ~5 SNPs per segment / 222093 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 3983 segments / ~6 SNPs per segment / 115292 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 9 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:46:12\n",
      "  * LOGfile : [shapeit_30102015_23h46m12s_44c676d1-d777-4898-a730-ab9a9d53e982.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446273972\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.map]\n",
      "  * 103 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.13.shapeitmap]\n",
      "  * 214 genetic positions found\n",
      "  * #set=103 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 97.19 Mb] / Genetic map [0.00 cM -> 97.18 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 78 individuals with high rates of missing data (>5%)\n",
      "  * 56 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 3432 segments / ~5 SNPs per segment / 168353 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 3120 segments / ~5 SNPs per segment / 87646 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 7 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:46:19\n",
      "  * LOGfile : [shapeit_30102015_23h46m19s_6f0238f9-d6e8-4840-a732-0a633d3a3d25.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446273979\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.map]\n",
      "  * 118 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.14.shapeitmap]\n",
      "  * 207 genetic positions found\n",
      "  * #set=118 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 64.33 Mb] / Genetic map [0.00 cM -> 64.32 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 74 individuals with high rates of missing data (>5%)\n",
      "  * 49 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 3825 segments / ~5 SNPs per segment / 181264 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 3466 segments / ~6 SNPs per segment / 98517 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 6 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:46:25\n",
      "  * LOGfile : [shapeit_30102015_23h46m25s_24842efe-7e87-421a-918e-0511802e7661.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446273985\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.map]\n",
      "  * 118 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.15.shapeitmap]\n",
      "  * 198 genetic positions found\n",
      "  * #set=118 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 92.28 Mb] / Genetic map [0.00 cM -> 92.27 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 82 individuals with high rates of missing data (>5%)\n",
      "  * 60 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 3857 segments / ~5 SNPs per segment / 188974 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 3576 segments / ~5 SNPs per segment / 102637 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 9 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:46:34\n",
      "  * LOGfile : [shapeit_30102015_23h46m34s_8017dbab-16c2-4cf9-a688-f8715de9e130.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446273994\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.map]\n",
      "  * 102 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.16.shapeitmap]\n",
      "  * 184 genetic positions found\n",
      "  * #set=102 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 89.57 Mb] / Genetic map [0.00 cM -> 89.56 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 71 individuals with high rates of missing data (>5%)\n",
      "  * 53 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 3329 segments / ~5 SNPs per segment / 165078 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 2914 segments / ~6 SNPs per segment / 80885 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 7 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:46:41\n",
      "  * LOGfile : [shapeit_30102015_23h46m41s_d8a714f1-fd31-4cc7-b04a-5cd302f0966b.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274001\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.map]\n",
      "  * 100 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.17.shapeitmap]\n",
      "  * 164 genetic positions found\n",
      "  * #set=100 / #interpolated=0\n",
      "  * Physical map [0.00 Mb -> 46.97 Mb] / Genetic map [0.00 cM -> 46.97 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 72 individuals with high rates of missing data (>5%)\n",
      "  * 50 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 3261 segments / ~5 SNPs per segment / 156436 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 3020 segments / ~5 SNPs per segment / 86067 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 5 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:46:46\n",
      "  * LOGfile : [shapeit_30102015_23h46m46s_bbc256bf-134a-4123-8376-fb98c8f3c04d.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274006\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.map]\n",
      "  * 111 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.18.shapeitmap]\n",
      "  * 182 genetic positions found\n",
      "  * #set=111 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 46.04 Mb] / Genetic map [0.00 cM -> 46.03 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 79 individuals with high rates of missing data (>5%)\n",
      "  * 52 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 3631 segments / ~5 SNPs per segment / 175219 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 3370 segments / ~5 SNPs per segment / 97762 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 6 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:46:52\n",
      "  * LOGfile : [shapeit_30102015_23h46m52s_800dc2a9-c663-46b2-bf45-d0821ec733c5.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274012\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.map]\n",
      "  * 119 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.19.shapeitmap]\n",
      "  * 206 genetic positions found\n",
      "  * #set=119 / #interpolated=0\n",
      "  * Physical map [4.71 Mb -> 60.19 Mb] / Genetic map [4.70 cM -> 60.18 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 83 individuals with high rates of missing data (>5%)\n",
      "  * 50 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 3772 segments / ~5 SNPs per segment / 181569 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 3415 segments / ~6 SNPs per segment / 102213 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 5 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:46:57\n",
      "  * LOGfile : [shapeit_30102015_23h46m57s_afe1f6d8-bf29-4d92-b074-b4edc96b6444.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274017\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.map]\n",
      "  * 93 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.20.shapeitmap]\n",
      "  * 175 genetic positions found\n",
      "  * #set=93 / #interpolated=0\n",
      "  * Physical map [0.00 Mb -> 85.29 Mb] / Genetic map [0.00 cM -> 85.29 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 84 individuals with high rates of missing data (>5%)\n",
      "  * 41 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 2970 segments / ~5 SNPs per segment / 144227 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 2665 segments / ~6 SNPs per segment / 73317 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 6 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:47:03\n",
      "  * LOGfile : [shapeit_30102015_23h47m03s_f7d4161a-8e9d-4428-9298-8da57c626e72.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274023\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.map]\n",
      "  * 101 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.21.shapeitmap]\n",
      "  * 175 genetic positions found\n",
      "  * #set=101 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 54.72 Mb] / Genetic map [0.00 cM -> 54.71 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 68 individuals with high rates of missing data (>5%)\n",
      "  * 46 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 3236 segments / ~5 SNPs per segment / 157479 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 2763 segments / ~6 SNPs per segment / 75472 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 6 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:47:09\n",
      "  * LOGfile : [shapeit_30102015_23h47m09s_a8b68dc1-e716-4a43-80a7-13ab0abea6ae.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274029\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.map]\n",
      "  * 101 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.22.shapeitmap]\n",
      "  * 185 genetic positions found\n",
      "  * #set=101 / #interpolated=0\n",
      "  * Physical map [3.00 Mb -> 47.24 Mb] / Genetic map [2.99 cM -> 47.23 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 75 individuals with high rates of missing data (>5%)\n",
      "  * 53 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 3286 segments / ~5 SNPs per segment / 163214 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 3069 segments / ~5 SNPs per segment / 87997 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 4 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:47:13\n",
      "  * LOGfile : [shapeit_30102015_23h47m13s_4960dafc-54c5-46ef-b607-4ad7af7261f0.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274033\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.map]\n",
      "  * 111 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.23.shapeitmap]\n",
      "  * 196 genetic positions found\n",
      "  * #set=111 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 49.23 Mb] / Genetic map [0.00 cM -> 49.22 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 64 individuals with high rates of missing data (>5%)\n",
      "  * 51 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 3573 segments / ~5 SNPs per segment / 164655 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 3376 segments / ~5 SNPs per segment / 96518 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 6 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:47:19\n",
      "  * LOGfile : [shapeit_30102015_23h47m19s_d6676023-10e1-47d3-b25d-af6761256fa5.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274039\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.map]\n",
      "  * 100 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.24.shapeitmap]\n",
      "  * 156 genetic positions found\n",
      "  * #set=100 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 59.38 Mb] / Genetic map [0.00 cM -> 59.37 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 65 individuals with high rates of missing data (>5%)\n",
      "  * 45 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 3270 segments / ~5 SNPs per segment / 155285 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 2867 segments / ~6 SNPs per segment / 82148 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 5 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:47:24\n",
      "  * LOGfile : [shapeit_30102015_23h47m24s_73707233-dbd7-4f4e-b58d-02406fd167b2.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274044\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.map]\n",
      "  * 82 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.25.shapeitmap]\n",
      "  * 167 genetic positions found\n",
      "  * #set=82 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 56.64 Mb] / Genetic map [0.00 cM -> 56.63 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 71 individuals with high rates of missing data (>5%)\n",
      "  * 48 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 2691 segments / ~5 SNPs per segment / 128148 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 2304 segments / ~6 SNPs per segment / 64266 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 5 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:47:29\n",
      "  * LOGfile : [shapeit_30102015_23h47m29s_4438317b-29d3-4878-8b5c-7670b5c688bb.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274049\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.map]\n",
      "  * 113 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.26.shapeitmap]\n",
      "  * 219 genetic positions found\n",
      "  * #set=113 / #interpolated=0\n",
      "  * Physical map [0.49 Mb -> 71.22 Mb] / Genetic map [0.48 cM -> 71.21 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 87 individuals with high rates of missing data (>5%)\n",
      "  * 54 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 3622 segments / ~5 SNPs per segment / 176660 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 3276 segments / ~6 SNPs per segment / 96283 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 7 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:47:36\n",
      "  * LOGfile : [shapeit_30102015_23h47m36s_6e7ad973-c892-4c03-92d1-7837e63e5d2f.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274056\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.map]\n",
      "  * 90 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.27.shapeitmap]\n",
      "  * 153 genetic positions found\n",
      "  * #set=90 / #interpolated=0\n",
      "  * Physical map [6.24 Mb -> 56.12 Mb] / Genetic map [6.23 cM -> 56.11 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 65 individuals with high rates of missing data (>5%)\n",
      "  * 41 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 2826 segments / ~5 SNPs per segment / 130875 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 2524 segments / ~6 SNPs per segment / 71359 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 3 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:47:39\n",
      "  * LOGfile : [shapeit_30102015_23h47m39s_fd6e74cc-01af-4488-b2d6-aed7d4019e12.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274059\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.map]\n",
      "  * 108 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.28.shapeitmap]\n",
      "  * 174 genetic positions found\n",
      "  * #set=108 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 63.10 Mb] / Genetic map [0.00 cM -> 63.09 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 78 individuals with high rates of missing data (>5%)\n",
      "  * 46 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 3389 segments / ~5 SNPs per segment / 159659 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 3109 segments / ~6 SNPs per segment / 89203 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 7 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:47:46\n",
      "  * LOGfile : [shapeit_30102015_23h47m46s_f97f22bc-fe92-4f04-84cf-486923eb1c59.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274066\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.map]\n",
      "  * 92 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.29.shapeitmap]\n",
      "  * 151 genetic positions found\n",
      "  * #set=92 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 59.79 Mb] / Genetic map [0.00 cM -> 59.78 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 70 individuals with high rates of missing data (>5%)\n",
      "  * 41 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 2947 segments / ~5 SNPs per segment / 136454 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 2707 segments / ~6 SNPs per segment / 75249 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 5 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:47:51\n",
      "  * LOGfile : [shapeit_30102015_23h47m51s_67ed475f-6337-43a4-84de-7561966324eb.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274071\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.map]\n",
      "  * 83 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.30.shapeitmap]\n",
      "  * 137 genetic positions found\n",
      "  * #set=83 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 50.27 Mb] / Genetic map [0.00 cM -> 50.26 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 79 individuals with high rates of missing data (>5%)\n",
      "  * 43 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 2738 segments / ~5 SNPs per segment / 130738 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 2352 segments / ~6 SNPs per segment / 64475 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 4 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:47:56\n",
      "  * LOGfile : [shapeit_30102015_23h47m56s_9e6ba952-7bb1-41ab-90c3-67898c9b4aac.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274076\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.map]\n",
      "  * 85 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.31.shapeitmap]\n",
      "  * 162 genetic positions found\n",
      "  * #set=85 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 54.51 Mb] / Genetic map [0.00 cM -> 54.50 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 72 individuals with high rates of missing data (>5%)\n",
      "  * 43 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 2701 segments / ~5 SNPs per segment / 132338 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 2545 segments / ~5 SNPs per segment / 71524 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 5 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:48:01\n",
      "  * LOGfile : [shapeit_30102015_23h48m01s_bcfa6956-a77e-414c-a98e-30b3edafb760.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274081\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.map]\n",
      "  * 90 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.32.shapeitmap]\n",
      "  * 161 genetic positions found\n",
      "  * #set=90 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 57.99 Mb] / Genetic map [0.00 cM -> 57.98 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 74 individuals with high rates of missing data (>5%)\n",
      "  * 46 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 2918 segments / ~5 SNPs per segment / 138630 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 2627 segments / ~6 SNPs per segment / 72665 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 5 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:48:06\n",
      "  * LOGfile : [shapeit_30102015_23h48m06s_6be334b6-962b-47f9-9068-7f8a9b592df8.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274086\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.map]\n",
      "  * 80 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.33.shapeitmap]\n",
      "  * 125 genetic positions found\n",
      "  * #set=80 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 36.79 Mb] / Genetic map [0.00 cM -> 36.78 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 74 individuals with high rates of missing data (>5%)\n",
      "  * 44 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 2611 segments / ~5 SNPs per segment / 123608 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 2201 segments / ~6 SNPs per segment / 61577 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 4 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:48:10\n",
      "  * LOGfile : [shapeit_30102015_23h48m10s_de85e9c9-043b-441d-b4ae-1e246291d328.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274090\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.map]\n",
      "  * 74 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.34.shapeitmap]\n",
      "  * 153 genetic positions found\n",
      "  * #set=74 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 51.91 Mb] / Genetic map [0.00 cM -> 51.90 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 83 individuals with high rates of missing data (>5%)\n",
      "  * 35 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 2411 segments / ~5 SNPs per segment / 114676 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 2024 segments / ~6 SNPs per segment / 55651 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 5 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:48:15\n",
      "  * LOGfile : [shapeit_30102015_23h48m15s_4c38f8b1-81bb-4419-ab60-1d885b41795c.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274095\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.map]\n",
      "  * 74 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.35.shapeitmap]\n",
      "  * 124 genetic positions found\n",
      "  * #set=74 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 61.13 Mb] / Genetic map [0.00 cM -> 61.12 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 83 individuals with high rates of missing data (>5%)\n",
      "  * 31 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 2375 segments / ~5 SNPs per segment / 109453 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 2124 segments / ~6 SNPs per segment / 58932 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 4 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:48:19\n",
      "  * LOGfile : [shapeit_30102015_23h48m19s_9dd2d405-fd19-4b58-9992-067b986b457f.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274099\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.map]\n",
      "  * 61 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.36.shapeitmap]\n",
      "  * 146 genetic positions found\n",
      "  * #set=61 / #interpolated=0\n",
      "  * Physical map [0.00 Mb -> 52.44 Mb] / Genetic map [0.00 cM -> 52.44 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 73 individuals with high rates of missing data (>5%)\n",
      "  * 32 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 2084 segments / ~5 SNPs per segment / 96557 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 1857 segments / ~5 SNPs per segment / 51473 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 4 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 23:48:24\n",
      "  * LOGfile : [shapeit_30102015_23h48m24s_c1f2623d-bff3-4693-8c72-461e5a62651d.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446274104\n",
      "  * Parallelisation: 7 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 70 iterations [10 B + 1 runs of 10 P + 50 M]\n",
      "  * Model: 10 states per window [10 H + 0 PM + 0 R + 0 COV ] / Windows of ~10.0 Mb / Ne = 6\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.map]\n",
      "  * 64 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.ped] where missing-code = [0]\n",
      "  * 177 samples included\n",
      "  * 0 unrelateds / 0 duos / 175 trios in 1 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.37.shapeitmap]\n",
      "  * 98 genetic positions found\n",
      "  * #set=64 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 56.29 Mb] / Genetic map [0.00 cM -> 56.28 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 75 individuals with high rates of missing data (>5%)\n",
      "  * 24 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [177/177]\n",
      "  * 177 graphs / 2016 segments / ~5 SNPs per segment / 91484 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [177/177]\n",
      "\n",
      "Burn-in iteration [1/10] [177/177]\n",
      "\n",
      "Burn-in iteration [2/10] [177/177]\n",
      "\n",
      "Burn-in iteration [3/10] [177/177]\n",
      "\n",
      "Burn-in iteration [4/10] [177/177]\n",
      "\n",
      "Burn-in iteration [5/10] [177/177]\n",
      "\n",
      "Burn-in iteration [6/10] [177/177]\n",
      "\n",
      "Burn-in iteration [7/10] [177/177]\n",
      "\n",
      "Burn-in iteration [8/10] [177/177]\n",
      "\n",
      "Burn-in iteration [9/10] [177/177]\n",
      "\n",
      "Burn-in iteration [10/10] [177/177]\n",
      "\n",
      "Pruning iteration [1/10] [177/177]\n",
      "\n",
      "Pruning iteration [2/10] [177/177]\n",
      "\n",
      "Pruning iteration [3/10] [177/177]\n",
      "\n",
      "Pruning iteration [4/10] [177/177]\n",
      "\n",
      "Pruning iteration [5/10] [177/177]\n",
      "\n",
      "Pruning iteration [6/10] [177/177]\n",
      "\n",
      "Pruning iteration [7/10] [177/177]\n",
      "\n",
      "Pruning iteration [8/10] [177/177]\n",
      "\n",
      "Pruning iteration [9/10] [177/177]\n",
      "\n",
      "Pruning iteration [10/10] [177/177]\n",
      "\n",
      "Pruning graphs [177/177]\n",
      "  * 177 graphs / 1759 segments / ~6 SNPs per segment / 48048 transitions\n",
      "  * 0 haploids / 177 unrelateds / 0 duos / 0 trios\n",
      "  * 354 founder haplotypes\n",
      "\n",
      "Main iteration [1/50] [177/177]\n",
      "\n",
      "Main iteration [2/50] [177/177]\n",
      "\n",
      "Main iteration [3/50] [177/177]\n",
      "\n",
      "Main iteration [4/50] [177/177]\n",
      "\n",
      "Main iteration [5/50] [177/177]\n",
      "\n",
      "Main iteration [6/50] [177/177]\n",
      "\n",
      "Main iteration [7/50] [177/177]\n",
      "\n",
      "Main iteration [8/50] [177/177]\n",
      "\n",
      "Main iteration [9/50] [177/177]\n",
      "\n",
      "Main iteration [10/50] [177/177]\n",
      "\n",
      "Main iteration [11/50] [177/177]\n",
      "\n",
      "Main iteration [12/50] [177/177]\n",
      "\n",
      "Main iteration [13/50] [177/177]\n",
      "\n",
      "Main iteration [14/50] [177/177]\n",
      "\n",
      "Main iteration [15/50] [177/177]\n",
      "\n",
      "Main iteration [16/50] [177/177]\n",
      "\n",
      "Main iteration [17/50] [177/177]\n",
      "\n",
      "Main iteration [18/50] [177/177]\n",
      "\n",
      "Main iteration [19/50] [177/177]\n",
      "\n",
      "Main iteration [20/50] [177/177]\n",
      "\n",
      "Main iteration [21/50] [177/177]\n",
      "\n",
      "Main iteration [22/50] [177/177]\n",
      "\n",
      "Main iteration [23/50] [177/177]\n",
      "\n",
      "Main iteration [24/50] [177/177]\n",
      "\n",
      "Main iteration [25/50] [177/177]\n",
      "\n",
      "Main iteration [26/50] [177/177]\n",
      "\n",
      "Main iteration [27/50] [177/177]\n",
      "\n",
      "Main iteration [28/50] [177/177]\n",
      "\n",
      "Main iteration [29/50] [177/177]\n",
      "\n",
      "Main iteration [30/50] [177/177]\n",
      "\n",
      "Main iteration [31/50] [177/177]\n",
      "\n",
      "Main iteration [32/50] [177/177]\n",
      "\n",
      "Main iteration [33/50] [177/177]\n",
      "\n",
      "Main iteration [34/50] [177/177]\n",
      "\n",
      "Main iteration [35/50] [177/177]\n",
      "\n",
      "Main iteration [36/50] [177/177]\n",
      "\n",
      "Main iteration [37/50] [177/177]\n",
      "\n",
      "Main iteration [38/50] [177/177]\n",
      "\n",
      "Main iteration [39/50] [177/177]\n",
      "\n",
      "Main iteration [40/50] [177/177]\n",
      "\n",
      "Main iteration [41/50] [177/177]\n",
      "\n",
      "Main iteration [42/50] [177/177]\n",
      "\n",
      "Main iteration [43/50] [177/177]\n",
      "\n",
      "Main iteration [44/50] [177/177]\n",
      "\n",
      "Main iteration [45/50] [177/177]\n",
      "\n",
      "Main iteration [46/50] [177/177]\n",
      "\n",
      "Main iteration [47/50] [177/177]\n",
      "\n",
      "Main iteration [48/50] [177/177]\n",
      "\n",
      "Main iteration [49/50] [177/177]\n",
      "\n",
      "Main iteration [50/50] [177/177]\n",
      "\n",
      "Normalising graphs [177/177]\n",
      "\n",
      "Solving haplotypes [177/177]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.phased.haps]\n",
      "\n",
      "Writing SNPs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Writing genotype graphs in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.phased.graph] in SHAPEIT binary format (v2.2)\n",
      "\n",
      "Running time: 4 seconds\n"
     ]
    }
   ],
   "source": [
    "for chrom in range(1, 38):\n",
    "    !'/home/ipseg/Programs/SHAPEIT/shapeit.v2.r837.GLIBCv2.12.Linux.static/bin/shapeit' \\\n",
    "    --input-ped {'/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom{}.ped'.format(chrom)} \\\n",
    "      {'/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom{}.map'.format(chrom)} \\\n",
    "    --duohmm \\\n",
    "    --states 100 --window 10 \\\n",
    "    --effective-size 1000 \\\n",
    "    --thread 7 \\\n",
    "    --burn 100 --prune 100 --main 500 \\\n",
    "    --input-map {'/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.{}.shapeitmap'.format(chrom)} \\\n",
    "    --output-max {'/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom{}.phased'.format(chrom)} \\\n",
    "    --output-graph {'/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom{}.phased.graph'.format(chrom)}\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Load phase data \n",
    "because of how we code genotypes for LEPmap - we want the sire phase:\n",
    "    - family 8- line 4 \n",
    "    - family 1- line 181\n",
    "    - family 9- line 217\n",
    "    \n",
    "4+2*(x-2) = (8,9) (362, 363), [434, 435]\n",
    "\n",
    "    \n",
    "this corresponds to columns:\n",
    "    - [7,8]\n",
    "    - [361, 362]\n",
    "    - [433, 434]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "aa = None\n",
    "for chrom in range(1, 38):\n",
    "    haplotypes = pd.read_csv('/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom{}.phased.haps'.format(chrom), \n",
    "           sep = ' ', header = None)\n",
    "    fam8 = haplotypes.iloc[:,[7,8]]\n",
    "    #fam1 = haplotypes.iloc[:,[361, 362]]\n",
    "    #fam9 = haplotypes.iloc[:,[433, 434]]\n",
    "\n",
    "    parental_haplotypes = fam8\n",
    "    #parental_haplotypes = pd.merge(pd.merge(fam8, fam1, right_index=True, left_index=True), fam9, right_index=True, left_index=True)\n",
    "    \n",
    "    phase8 = parental_haplotypes[7] > parental_haplotypes[8]\n",
    "    #phase1 = parental_haplotypes[361] > parental_haplotypes[362]\n",
    "    #phase9 = parental_haplotypes[433] > parental_haplotypes[434]\n",
    "    \n",
    "    LEPnames = pd.read_csv('/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom{}.map'.format(chrom), \n",
    "                           sep = '\\t', header = None)\n",
    "\n",
    "    LEPnames = LEPnames[1].values.tolist()\n",
    "    #print chrom, LEPnames[-1]\n",
    "    current_phase = pd.DataFrame({'LEPname':LEPnames, \n",
    "                                  'fam_08':phase8\n",
    "                                  #'fam_01':phase1, \n",
    "                                  #'fam_09':phase9\n",
    "                                 } )\n",
    "    if aa is None:\n",
    "        aa = current_phase\n",
    "    else:\n",
    "        aa = aa.append(current_phase)\n",
    "parental_phases = aa\n",
    "    \n",
    "    \n",
    "    \n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "parental_phases.to_csv('/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/parental_phase.txt', \n",
    "                       sep = '\\t', index = False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "aa.tail(50)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "chr9_phased = pd.read_csv('/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom{}.map'.format(chrom), \n",
    "           sep = ' ', header = None)\n",
    "\n",
    "fam8 = chr9_phased.iloc[:,[7,8]]\n",
    "fam1 = chr9_phased.iloc[:,[361, 362]]\n",
    "fam9 = chr9_phased.iloc[:,[433, 434]]\n",
    "\n",
    "parental_haplotypes = pd.merge(pd.merge(fam8, fam1, right_index=True, left_index=True), fam9, right_index=True, left_index=True)\n",
    "parental_haplotypes.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "phase8 = parental_haplotypes[7] > parental_haplotypes[8]\n",
    "phase1 = parental_haplotypes[361] > parental_haplotypes[362]\n",
    "phase9 = parental_haplotypes[433] > parental_haplotypes[434]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "lmap"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# For each locus within each family:\n",
    "if the phase is False:\n",
    "switch the inferred phase of all offspring "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "phase9"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "pd.merge(chr9_phased.iloc[:,[8,9]], chr9_phased.iloc[:,[362,363]], left_index=True, right_index=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
