{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%matplotlib inline\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "from IPython.core.pylabtools import figsize\n",
    "import matplotlib.pyplot as plt\n",
    "import os.path\n",
    "import pandas as pd\n",
    "import collections\n",
    "import seaborn as sns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>contig</th>\n",
       "      <th>resolved_locus</th>\n",
       "      <th>stacks_CatID</th>\n",
       "      <th>stacks_SNP</th>\n",
       "      <th>LEPname</th>\n",
       "      <th>LEP_LG</th>\n",
       "      <th>cM</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>6988</th>\n",
       "      <td>c2953</td>\n",
       "      <td>2953_C</td>\n",
       "      <td>16919</td>\n",
       "      <td>16919_25</td>\n",
       "      <td>2902</td>\n",
       "      <td>37</td>\n",
       "      <td>56.28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6989</th>\n",
       "      <td>c56816</td>\n",
       "      <td>56816_C</td>\n",
       "      <td>39923</td>\n",
       "      <td>39923_44</td>\n",
       "      <td>7366</td>\n",
       "      <td>37</td>\n",
       "      <td>56.28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6990</th>\n",
       "      <td>c47181</td>\n",
       "      <td>47181_C</td>\n",
       "      <td>31760</td>\n",
       "      <td>31760_82</td>\n",
       "      <td>5379</td>\n",
       "      <td>37</td>\n",
       "      <td>56.28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6991</th>\n",
       "      <td>c7090</td>\n",
       "      <td>7090_A</td>\n",
       "      <td>51610</td>\n",
       "      <td>51610_40</td>\n",
       "      <td>10169</td>\n",
       "      <td>37</td>\n",
       "      <td>56.28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6992</th>\n",
       "      <td>c56688</td>\n",
       "      <td>56688_C</td>\n",
       "      <td>39817</td>\n",
       "      <td>39817_67</td>\n",
       "      <td>7337</td>\n",
       "      <td>37</td>\n",
       "      <td>66.16</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      contig resolved_locus  stacks_CatID stacks_SNP  LEPname  LEP_LG     cM\n",
       "6988   c2953         2953_C         16919   16919_25     2902      37  56.28\n",
       "6989  c56816        56816_C         39923   39923_44     7366      37  56.28\n",
       "6990  c47181        47181_C         31760   31760_82     5379      37  56.28\n",
       "6991   c7090         7090_A         51610   51610_40    10169      37  56.28\n",
       "6992  c56688        56688_C         39817   39817_67     7337      37  66.16"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lmap = pd.read_csv(os.path.join('linkage_map','LEPmap','with_paralogs','final','PS_chum_map_2015.txt'), sep = '\\t')\n",
    "lmap = lmap.drop(['cM_OLD', 'paper1_LG'], axis = 1)\n",
    "lmap= lmap.drop_duplicates('LEPname')\n",
    "lmap = lmap.sort_values(['LEP_LG', 'cM'])\n",
    "lmap.tail()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "lmap['fakeBP'] = (lmap['cM']*1000000 + lmap.index).astype(np.int)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>contig</th>\n",
       "      <th>resolved_locus</th>\n",
       "      <th>stacks_CatID</th>\n",
       "      <th>stacks_SNP</th>\n",
       "      <th>LEPname</th>\n",
       "      <th>LEP_LG</th>\n",
       "      <th>cM</th>\n",
       "      <th>fakeBP</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>6988</th>\n",
       "      <td>c2953</td>\n",
       "      <td>2953_C</td>\n",
       "      <td>16919</td>\n",
       "      <td>16919_25</td>\n",
       "      <td>2902</td>\n",
       "      <td>37</td>\n",
       "      <td>56.28</td>\n",
       "      <td>56286988</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6989</th>\n",
       "      <td>c56816</td>\n",
       "      <td>56816_C</td>\n",
       "      <td>39923</td>\n",
       "      <td>39923_44</td>\n",
       "      <td>7366</td>\n",
       "      <td>37</td>\n",
       "      <td>56.28</td>\n",
       "      <td>56286989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6990</th>\n",
       "      <td>c47181</td>\n",
       "      <td>47181_C</td>\n",
       "      <td>31760</td>\n",
       "      <td>31760_82</td>\n",
       "      <td>5379</td>\n",
       "      <td>37</td>\n",
       "      <td>56.28</td>\n",
       "      <td>56286990</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6991</th>\n",
       "      <td>c7090</td>\n",
       "      <td>7090_A</td>\n",
       "      <td>51610</td>\n",
       "      <td>51610_40</td>\n",
       "      <td>10169</td>\n",
       "      <td>37</td>\n",
       "      <td>56.28</td>\n",
       "      <td>56286991</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6992</th>\n",
       "      <td>c56688</td>\n",
       "      <td>56688_C</td>\n",
       "      <td>39817</td>\n",
       "      <td>39817_67</td>\n",
       "      <td>7337</td>\n",
       "      <td>37</td>\n",
       "      <td>66.16</td>\n",
       "      <td>66166992</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      contig resolved_locus  stacks_CatID stacks_SNP  LEPname  LEP_LG     cM  \\\n",
       "6988   c2953         2953_C         16919   16919_25     2902      37  56.28   \n",
       "6989  c56816        56816_C         39923   39923_44     7366      37  56.28   \n",
       "6990  c47181        47181_C         31760   31760_82     5379      37  56.28   \n",
       "6991   c7090         7090_A         51610   51610_40    10169      37  56.28   \n",
       "6992  c56688        56688_C         39817   39817_67     7337      37  66.16   \n",
       "\n",
       "        fakeBP  \n",
       "6988  56286988  \n",
       "6989  56286989  \n",
       "6990  56286990  \n",
       "6991  56286991  \n",
       "6992  66166992  "
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lmap.tail()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "mapfile = lmap[['LEP_LG', 'LEPname', 'cM', 'fakeBP']]\n",
    "mapfile.to_csv('/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.map', \n",
    "               sep = '\\t', index = False, header = False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>LEP_LG</th>\n",
       "      <th>fakeBP</th>\n",
       "      <th>rrate</th>\n",
       "      <th>cM</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1</td>\n",
       "      <td>2</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>1</td>\n",
       "      <td>6</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>1</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>1</td>\n",
       "      <td>14</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    LEP_LG  fakeBP  rrate  cM\n",
       "0        1       0      1   0\n",
       "2        1       2      1   0\n",
       "6        1       6      1   0\n",
       "12       1      12      1   0\n",
       "14       1      14      1   0"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "shapeit_mapfile = lmap[['LEP_LG','fakeBP', 'cM']].copy()\n",
    "shapeit_mapfile['rrate'] = 1.0\n",
    "shapeit_mapfile = shapeit_mapfile[['LEP_LG', 'fakeBP', 'rrate', 'cM']].copy()\n",
    "shapeit_mapfile.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "gb = shapeit_mapfile.groupby('LEP_LG')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "for xx in range(1, 38):\n",
    "    to_write = gb.get_group(xx).drop('LEP_LG', axis = 1)\n",
    "    to_write.to_csv('linkage_map/centromere_placement/shapeit/collapsed_loci.{}.shapeitmap'.format(xx), sep = ' ', header = True, index = False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#family</th>\n",
       "      <th>name</th>\n",
       "      <th>sire</th>\n",
       "      <th>dam</th>\n",
       "      <th>sex</th>\n",
       "      <th>blank</th>\n",
       "      <th>10001</th>\n",
       "      <th>10003_A</th>\n",
       "      <th>1000_C</th>\n",
       "      <th>10012</th>\n",
       "      <th>...</th>\n",
       "      <th>Oke_ndub3-58</th>\n",
       "      <th>Oke_pnrc2-78</th>\n",
       "      <th>Oke_psmd9-57</th>\n",
       "      <th>Oke_rab5a-117</th>\n",
       "      <th>Oke_sylc-90</th>\n",
       "      <th>Oke_thic-84</th>\n",
       "      <th>Oke_txnrd1-74</th>\n",
       "      <th>Oke_u0602-244</th>\n",
       "      <th>Oke_u217-172</th>\n",
       "      <th>Oke_zn593-152</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>fam_08_Dam</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>2</td>\n",
       "      <td>0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>...</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>fam_08_Sire</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>...</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>CMUW10X08H_0002</td>\n",
       "      <td>fam_08_Sire</td>\n",
       "      <td>fam_08_Dam</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>...</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>CMUW10X08H_0003</td>\n",
       "      <td>fam_08_Sire</td>\n",
       "      <td>fam_08_Dam</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 1</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>...</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>CMUW10X08H_0005</td>\n",
       "      <td>fam_08_Sire</td>\n",
       "      <td>fam_08_Dam</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>...</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 12157 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "  #family             name         sire         dam  sex  blank 10001 10003_A  \\\n",
       "0  fam_08       fam_08_Dam            0           0    2      0   1 1     1 1   \n",
       "1  fam_08      fam_08_Sire            0           0    1      0   1 2     1 2   \n",
       "2  fam_08  CMUW10X08H_0002  fam_08_Sire  fam_08_Dam    0      0   1 1     1 2   \n",
       "3  fam_08  CMUW10X08H_0003  fam_08_Sire  fam_08_Dam    0      0   1 2     1 1   \n",
       "4  fam_08  CMUW10X08H_0005  fam_08_Sire  fam_08_Dam    0      0   1 2     1 2   \n",
       "\n",
       "  1000_C 10012      ...      Oke_ndub3-58 Oke_pnrc2-78 Oke_psmd9-57  \\\n",
       "0    1 1   1 1      ...               1 1          1 1          1 1   \n",
       "1    1 2   1 2      ...               1 2          1 2          1 2   \n",
       "2    0 0   0 0      ...               0 0          1 2          0 0   \n",
       "3    0 0   0 0      ...               0 0          1 2          0 0   \n",
       "4    0 0   0 0      ...               0 0          1 2          0 0   \n",
       "\n",
       "  Oke_rab5a-117 Oke_sylc-90 Oke_thic-84 Oke_txnrd1-74 Oke_u0602-244  \\\n",
       "0           1 1         1 1         1 1           1 1           1 1   \n",
       "1           1 2         1 2         1 2           1 2           1 2   \n",
       "2           0 0         1 1         1 2           1 2           1 1   \n",
       "3           0 0         1 1         1 1           1 2           1 2   \n",
       "4           0 0         1 1         1 2           1 2           1 2   \n",
       "\n",
       "  Oke_u217-172 Oke_zn593-152  \n",
       "0          1 1           1 1  \n",
       "1          1 2           1 2  \n",
       "2          1 1           1 2  \n",
       "3          1 1           1 1  \n",
       "4          1 2           1 2  \n",
       "\n",
       "[5 rows x 12157 columns]"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lepmap_infile = pd.read_csv(os.path.join('linkage_map','LEPmap','with_paralogs', 'collapsed_loci.lepmap'), sep = '\\t')\n",
    "lepmap_infile.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "lepmap_infile.columns = lepmap_infile.columns[:6].tolist() + list(range(1,len(lepmap_infile.columns[6:])+1))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## subset and reorder base on linkage map"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#family</th>\n",
       "      <th>name</th>\n",
       "      <th>sire</th>\n",
       "      <th>dam</th>\n",
       "      <th>sex</th>\n",
       "      <th>blank</th>\n",
       "      <th>4584</th>\n",
       "      <th>7377</th>\n",
       "      <th>6722</th>\n",
       "      <th>1923</th>\n",
       "      <th>...</th>\n",
       "      <th>4552</th>\n",
       "      <th>6213</th>\n",
       "      <th>2558</th>\n",
       "      <th>5198</th>\n",
       "      <th>5942</th>\n",
       "      <th>2902</th>\n",
       "      <th>7366</th>\n",
       "      <th>5379</th>\n",
       "      <th>10169</th>\n",
       "      <th>7337</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>fam_08_Dam</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>2</td>\n",
       "      <td>0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>...</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>fam_08_Sire</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>...</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>CMUW10X08H_0002</td>\n",
       "      <td>fam_08_Sire</td>\n",
       "      <td>fam_08_Dam</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>0 0</td>\n",
       "      <td>...</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>0 0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>CMUW10X08H_0003</td>\n",
       "      <td>fam_08_Sire</td>\n",
       "      <td>fam_08_Dam</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>0 0</td>\n",
       "      <td>...</td>\n",
       "      <td>1 2</td>\n",
       "      <td>1 1</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>fam_08</td>\n",
       "      <td>CMUW10X08H_0005</td>\n",
       "      <td>fam_08_Sire</td>\n",
       "      <td>fam_08_Dam</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 1</td>\n",
       "      <td>0 0</td>\n",
       "      <td>...</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>1 2</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>0 0</td>\n",
       "      <td>1 1</td>\n",
       "      <td>0 0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 7801 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "  #family             name         sire         dam  sex  blank 4584 7377  \\\n",
       "0  fam_08       fam_08_Dam            0           0    2      0  1 1  1 1   \n",
       "1  fam_08      fam_08_Sire            0           0    1      0  1 2  1 2   \n",
       "2  fam_08  CMUW10X08H_0002  fam_08_Sire  fam_08_Dam    0      0  1 1  1 1   \n",
       "3  fam_08  CMUW10X08H_0003  fam_08_Sire  fam_08_Dam    0      0  1 1  1 1   \n",
       "4  fam_08  CMUW10X08H_0005  fam_08_Sire  fam_08_Dam    0      0  1 1  1 1   \n",
       "\n",
       "  6722 1923 ...  4552 6213 2558 5198 5942 2902 7366 5379 10169 7337  \n",
       "0  1 1  1 1 ...   1 1  1 1  1 1  1 1  1 1  1 1  1 1  1 1   1 1  1 1  \n",
       "1  1 2  1 2 ...   1 2  1 2  1 2  1 2  1 2  1 2  1 2  1 2   1 2  1 2  \n",
       "2  1 1  0 0 ...   1 1  1 2  0 0  1 1  1 2  0 0  0 0  0 0   1 1  0 0  \n",
       "3  1 1  0 0 ...   1 2  1 1  0 0  0 0  1 1  0 0  0 0  0 0   1 2  0 0  \n",
       "4  1 1  0 0 ...   1 1  1 2  0 0  1 1  1 2  0 0  0 0  0 0   1 1  0 0  \n",
       "\n",
       "[5 rows x 7801 columns]"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ped = lepmap_infile[lepmap_infile.columns[:6].tolist() + lmap['LEPname'].tolist()]\n",
    "ped.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "ped.to_csv('/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.ped', \n",
    "               sep = '\\t', index = False, header = False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Split by chromosome in PLINK"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 1\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (331 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1-temporary.fam\n",
      "written.\n",
      "331 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.553472.\n",
      "331 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 2\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (300 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2-temporary.fam\n",
      "written.\n",
      "300 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.588943.\n",
      "300 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 3\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (320 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3-temporary.fam\n",
      "written.\n",
      "320 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.545617.\n",
      "320 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 4\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (270 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4-temporary.fam\n",
      "written.\n",
      "270 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.555736.\n",
      "270 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 5\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (369 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5-temporary.fam\n",
      "written.\n",
      "369 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.454018.\n",
      "369 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 6\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (320 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6-temporary.fam\n",
      "written.\n",
      "320 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.481225.\n",
      "320 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 7\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (268 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7-temporary.fam\n",
      "written.\n",
      "268 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.515198.\n",
      "268 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 8\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (282 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8-temporary.fam\n",
      "written.\n",
      "282 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.522415.\n",
      "282 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 9\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (286 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9-temporary.fam\n",
      "written.\n",
      "286 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.495935.\n",
      "286 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 10\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (274 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10-temporary.fam\n",
      "written.\n",
      "274 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.506869.\n",
      "274 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 11\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (303 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11-temporary.fam\n",
      "written.\n",
      "303 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.465534.\n",
      "303 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 12\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (260 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12-temporary.fam\n",
      "written.\n",
      "260 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.505191.\n",
      "260 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 13\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (214 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13-temporary.fam\n",
      "written.\n",
      "214 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.469778.\n",
      "214 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 14\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (207 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14-temporary.fam\n",
      "written.\n",
      "207 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.509642.\n",
      "207 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 15\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (198 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15-temporary.fam\n",
      "written.\n",
      "198 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.526731.\n",
      "198 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 16\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (184 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16-temporary.fam\n",
      "written.\n",
      "184 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.503248.\n",
      "184 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 17\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (164 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17-temporary.fam\n",
      "written.\n",
      "164 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.529942.\n",
      "164 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 18\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (182 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18-temporary.fam\n",
      "written.\n",
      "182 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.541119.\n",
      "182 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 19\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (206 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19-temporary.fam\n",
      "written.\n",
      "206 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.502921.\n",
      "206 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 20\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (175 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20-temporary.fam\n",
      "written.\n",
      "175 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.498281.\n",
      "175 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 21\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (175 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21-temporary.fam\n",
      "written.\n",
      "175 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.526574.\n",
      "175 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 22\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (185 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22-temporary.fam\n",
      "written.\n",
      "185 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.494858.\n",
      "185 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 23\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (196 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23-temporary.fam\n",
      "written.\n",
      "196 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.529949.\n",
      "196 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 24\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (156 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24-temporary.fam\n",
      "written.\n",
      "156 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.550709.\n",
      "156 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 25\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (167 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25-temporary.fam\n",
      "written.\n",
      "167 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.471788.\n",
      "167 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 26\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (219 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26-temporary.fam\n",
      "written.\n",
      "219 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.480677.\n",
      "219 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 27\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (153 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27-temporary.fam\n",
      "written.\n",
      "153 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.537914.\n",
      "153 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 28\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (174 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28-temporary.fam\n",
      "written.\n",
      "174 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.543337.\n",
      "174 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 29\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (151 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29-temporary.fam\n",
      "written.\n",
      "151 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.548484.\n",
      "151 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 30\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (137 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30-temporary.fam\n",
      "written.\n",
      "137 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.535013.\n",
      "137 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 31\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (162 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31-temporary.fam\n",
      "written.\n",
      "162 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.49202.\n",
      "162 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 32\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (161 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32-temporary.fam\n",
      "written.\n",
      "161 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.504494.\n",
      "161 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 33\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (125 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33-temporary.fam\n",
      "written.\n",
      "125 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.555252.\n",
      "125 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 34\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (153 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34-temporary.fam\n",
      "written.\n",
      "153 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.461927.\n",
      "153 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 35\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (124 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35-temporary.fam\n",
      "written.\n",
      "124 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.512457.\n",
      "124 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 36\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (146 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36-temporary.fam\n",
      "written.\n",
      "146 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.427303.\n",
      "146 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n",
      "PLINK v1.90b3w 64-bit (3 Sep 2015)         https://www.cog-genomics.org/plink2\n",
      "(C) 2005-2015 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.log.\n",
      "Options in effect:\n",
      "  --autosome-num 50\n",
      "  --chr 37\n",
      "  --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci\n",
      "  --out /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37\n",
      "  --recode\n",
      "\n",
      "32127 MB RAM detected; reserving 16063 MB for main workspace.\n",
      ".ped scan complete (for binary autoconversion).\n",
      "Performing single-pass .bed write (98 variants, 246 samples).\n",
      "--file:\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37-temporary.bed\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37-temporary.bim\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37-temporary.fam\n",
      "written.\n",
      "98 variants loaded from .bim file.\n",
      "246 samples (3 males, 3 females, 240 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.nosex\n",
      ".\n",
      "Using 1 thread (no multithreaded calculations invoked.\n",
      "Before main variant filters, 6 founders and 240 nonfounders present.\n",
      "Calculating allele frequencies... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\b\b done.\n",
      "Total genotyping rate is 0.559275.\n",
      "98 variants and 246 samples pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--recode to\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.ped\n",
      "+\n",
      "/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.map\n",
      "... 0%\b\b1%\b\b2%\b\b3%\b\b4%\b\b5%\b\b6%\b\b7%\b\b8%\b\b9%\b\b10%\b\b\b11%\b\b\b12%\b\b\b13%\b\b\b14%\b\b\b15%\b\b\b16%\b\b\b17%\b\b\b18%\b\b\b19%\b\b\b20%\b\b\b21%\b\b\b22%\b\b\b23%\b\b\b24%\b\b\b25%\b\b\b26%\b\b\b27%\b\b\b28%\b\b\b29%\b\b\b30%\b\b\b31%\b\b\b32%\b\b\b33%\b\b\b34%\b\b\b35%\b\b\b36%\b\b\b37%\b\b\b38%\b\b\b39%\b\b\b40%\b\b\b41%\b\b\b42%\b\b\b43%\b\b\b44%\b\b\b45%\b\b\b46%\b\b\b47%\b\b\b48%\b\b\b49%\b\b\b50%\b\b\b51%\b\b\b52%\b\b\b53%\b\b\b54%\b\b\b55%\b\b\b56%\b\b\b57%\b\b\b58%\b\b\b59%\b\b\b60%\b\b\b61%\b\b\b62%\b\b\b63%\b\b\b64%\b\b\b65%\b\b\b66%\b\b\b67%\b\b\b68%\b\b\b69%\b\b\b70%\b\b\b71%\b\b\b72%\b\b\b73%\b\b\b74%\b\b\b75%\b\b\b76%\b\b\b77%\b\b\b78%\b\b\b79%\b\b\b80%\b\b\b81%\b\b\b82%\b\b\b83%\b\b\b84%\b\b\b85%\b\b\b86%\b\b\b87%\b\b\b88%\b\b\b89%\b\b\b90%\b\b\b91%\b\b\b92%\b\b\b93%\b\b\b94%\b\b\b95%\b\b\b96%\b\b\b97%\b\b\b98%\b\b\b99%\b\b\bdone.\n"
     ]
    }
   ],
   "source": [
    "for chrom in range(1, 38):\n",
    "    !plink --file /home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci \\\n",
    "           --autosome-num 50 --chr {chrom} --recode \\\n",
    "           --out {'/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom{}'.format(chrom)}"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Run SHAPEIT2 to phase"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:19:03\n",
      "  * LOGfile : [shapeit_30102015_18h19m03s_6b2f742a-463c-42e2-bc45-452ab1f9688a.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446254343\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.map]\n",
      "  * 331 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.1.shapeitmap]\n",
      "  * 331 genetic positions found\n",
      "  * #set=331 / #interpolated=0\n",
      "  * Physical map [0.00 Mb -> 106.08 Mb] / Genetic map [0.00 cM -> 106.08 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 326 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 25595 segments / ~3 SNPs per segment / 2494037 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 15201 segments / ~5 SNPs per segment / 356239 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom1.phased.haps]\n",
      "\n",
      "Running time: 76 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:20:19\n",
      "  * LOGfile : [shapeit_30102015_18h20m19s_d682ec9c-c850-4336-a9a9-89f6b001fc63.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446254419\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.map]\n",
      "  * 300 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.2.shapeitmap]\n",
      "  * 300 genetic positions found\n",
      "  * #set=300 / #interpolated=0\n",
      "  * Physical map [0.00 Mb -> 111.35 Mb] / Genetic map [0.00 cM -> 111.35 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 291 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 22198 segments / ~3 SNPs per segment / 2095763 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 13324 segments / ~5 SNPs per segment / 309473 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom2.phased.haps]\n",
      "\n",
      "Running time: 80 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:21:39\n",
      "  * LOGfile : [shapeit_30102015_18h21m39s_1e23d776-459b-4a44-92f8-c7041514e7ec.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446254499\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.map]\n",
      "  * 320 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.3.shapeitmap]\n",
      "  * 320 genetic positions found\n",
      "  * #set=320 / #interpolated=0\n",
      "  * Physical map [2.65 Mb -> 129.59 Mb] / Genetic map [2.65 cM -> 129.59 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 303 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 25137 segments / ~3 SNPs per segment / 2379526 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 14632 segments / ~5 SNPs per segment / 340242 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom3.phased.haps]\n",
      "\n",
      "Running time: 82 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:23:02\n",
      "  * LOGfile : [shapeit_30102015_18h23m02s_7b2c7086-bdc0-44dd-90b6-59317eb03514.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446254582\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.map]\n",
      "  * 270 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.4.shapeitmap]\n",
      "  * 270 genetic positions found\n",
      "  * #set=270 / #interpolated=0\n",
      "  * Physical map [0.00 Mb -> 119.48 Mb] / Genetic map [0.00 cM -> 119.48 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 257 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 20861 segments / ~3 SNPs per segment / 1991304 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 12664 segments / ~5 SNPs per segment / 289286 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom4.phased.haps]\n",
      "\n",
      "Running time: 73 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:24:15\n",
      "  * LOGfile : [shapeit_30102015_18h24m15s_b6124dba-2ed1-4715-93df-cff9cef6e96b.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446254655\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.map]\n",
      "  * 369 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.5.shapeitmap]\n",
      "  * 369 genetic positions found\n",
      "  * #set=369 / #interpolated=0\n",
      "  * Physical map [0.00 Mb -> 137.46 Mb] / Genetic map [0.00 cM -> 137.46 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 360 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 31937 segments / ~2 SNPs per segment / 3272590 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 16474 segments / ~5 SNPs per segment / 369431 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom5.phased.haps]\n",
      "\n",
      "Running time: 93 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:25:48\n",
      "  * LOGfile : [shapeit_30102015_18h25m48s_2aef7eca-76c1-43a1-8ae4-8b7c3fd06b09.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446254748\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.map]\n",
      "  * 320 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.6.shapeitmap]\n",
      "  * 320 genetic positions found\n",
      "  * #set=320 / #interpolated=0\n",
      "  * Physical map [0.00 Mb -> 131.47 Mb] / Genetic map [0.00 cM -> 131.47 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 308 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 26719 segments / ~2 SNPs per segment / 2717885 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 14247 segments / ~5 SNPs per segment / 316835 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom6.phased.haps]\n",
      "\n",
      "Running time: 82 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:27:10\n",
      "  * LOGfile : [shapeit_30102015_18h27m10s_696cef2b-3fd9-4fa1-a760-9747522b146c.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446254830\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.map]\n",
      "  * 268 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.7.shapeitmap]\n",
      "  * 268 genetic positions found\n",
      "  * #set=268 / #interpolated=0\n",
      "  * Physical map [3.81 Mb -> 108.91 Mb] / Genetic map [3.80 cM -> 108.90 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 260 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 21619 segments / ~3 SNPs per segment / 2149804 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 11448 segments / ~5 SNPs per segment / 255534 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom7.phased.haps]\n",
      "\n",
      "Running time: 67 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:28:17\n",
      "  * LOGfile : [shapeit_30102015_18h28m17s_ffa8b1ac-9f1b-4b6c-8cfc-b91139440a5d.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446254897\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.map]\n",
      "  * 282 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.8.shapeitmap]\n",
      "  * 282 genetic positions found\n",
      "  * #set=282 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 121.59 Mb] / Genetic map [0.00 cM -> 121.58 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 272 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 22544 segments / ~3 SNPs per segment / 2245274 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 11980 segments / ~5 SNPs per segment / 259507 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom8.phased.haps]\n",
      "\n",
      "Running time: 74 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:29:31\n",
      "  * LOGfile : [shapeit_30102015_18h29m31s_08577bf3-4fa6-41fa-a03e-92a003b55be7.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446254971\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.map]\n",
      "  * 286 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.9.shapeitmap]\n",
      "  * 286 genetic positions found\n",
      "  * #set=286 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 113.91 Mb] / Genetic map [0.00 cM -> 113.90 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 279 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 23545 segments / ~2 SNPs per segment / 2399144 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 11183 segments / ~6 SNPs per segment / 245921 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.phased.haps]\n",
      "\n",
      "Running time: 72 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:30:43\n",
      "  * LOGfile : [shapeit_30102015_18h30m43s_11353e6f-563c-4c69-919a-438b226f6991.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255043\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.map]\n",
      "  * 274 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.10.shapeitmap]\n",
      "  * 274 genetic positions found\n",
      "  * #set=274 / #interpolated=0\n",
      "  * Physical map [7.05 Mb -> 127.00 Mb] / Genetic map [7.05 cM -> 127.00 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 261 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 22395 segments / ~3 SNPs per segment / 2191455 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 11374 segments / ~5 SNPs per segment / 251003 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom10.phased.haps]\n",
      "\n",
      "Running time: 68 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:31:51\n",
      "  * LOGfile : [shapeit_30102015_18h31m51s_b4b5064e-2e09-4240-9ba4-c30bed79729b.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255111\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.map]\n",
      "  * 303 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.11.shapeitmap]\n",
      "  * 303 genetic positions found\n",
      "  * #set=303 / #interpolated=0\n",
      "  * Physical map [0.00 Mb -> 139.53 Mb] / Genetic map [0.00 cM -> 139.53 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 295 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 25779 segments / ~2 SNPs per segment / 2584679 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 12509 segments / ~5 SNPs per segment / 268052 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom11.phased.haps]\n",
      "\n",
      "Running time: 79 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:33:10\n",
      "  * LOGfile : [shapeit_30102015_18h33m10s_5f1dffbf-6252-4efa-a7ca-66c42b4ae67e.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255190\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.map]\n",
      "  * 260 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.12.shapeitmap]\n",
      "  * 260 genetic positions found\n",
      "  * #set=260 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 127.12 Mb] / Genetic map [0.00 cM -> 127.11 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 251 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 21234 segments / ~3 SNPs per segment / 2061044 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 11567 segments / ~5 SNPs per segment / 250907 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom12.phased.haps]\n",
      "\n",
      "Running time: 70 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:34:20\n",
      "  * LOGfile : [shapeit_30102015_18h34m20s_444e8dfa-1c07-429f-8d5d-60690aa37afd.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255260\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.map]\n",
      "  * 214 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.13.shapeitmap]\n",
      "  * 214 genetic positions found\n",
      "  * #set=214 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 97.19 Mb] / Genetic map [0.00 cM -> 97.18 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 208 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 18241 segments / ~2 SNPs per segment / 1844589 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 9577 segments / ~5 SNPs per segment / 212389 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom13.phased.haps]\n",
      "\n",
      "Running time: 55 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:35:15\n",
      "  * LOGfile : [shapeit_30102015_18h35m15s_40729465-9e29-4458-b3df-adad9feaecc6.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255315\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.map]\n",
      "  * 207 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.14.shapeitmap]\n",
      "  * 207 genetic positions found\n",
      "  * #set=207 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 64.33 Mb] / Genetic map [0.00 cM -> 64.32 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 203 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 16789 segments / ~3 SNPs per segment / 1695188 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 9438 segments / ~5 SNPs per segment / 224109 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom14.phased.haps]\n",
      "\n",
      "Running time: 40 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:35:55\n",
      "  * LOGfile : [shapeit_30102015_18h35m55s_048410dd-ada7-4a65-a566-8cc20b9ea605.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255355\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.map]\n",
      "  * 198 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.15.shapeitmap]\n",
      "  * 198 genetic positions found\n",
      "  * #set=198 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 92.28 Mb] / Genetic map [0.00 cM -> 92.27 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 190 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 15888 segments / ~3 SNPs per segment / 1514214 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 9450 segments / ~5 SNPs per segment / 212055 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom15.phased.haps]\n",
      "\n",
      "Running time: 55 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:36:50\n",
      "  * LOGfile : [shapeit_30102015_18h36m50s_ae4a1ad1-ecb9-4b9b-9356-c1f12e6abf8f.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255410\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.map]\n",
      "  * 184 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.16.shapeitmap]\n",
      "  * 184 genetic positions found\n",
      "  * #set=184 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 89.57 Mb] / Genetic map [0.00 cM -> 89.56 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 180 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 15269 segments / ~2 SNPs per segment / 1442092 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 8145 segments / ~5 SNPs per segment / 175250 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom16.phased.haps]\n",
      "\n",
      "Running time: 51 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:37:41\n",
      "  * LOGfile : [shapeit_30102015_18h37m41s_415ad1f7-cf89-4f2e-b914-c26dbfc0af66.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255461\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.map]\n",
      "  * 164 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.17.shapeitmap]\n",
      "  * 164 genetic positions found\n",
      "  * #set=164 / #interpolated=0\n",
      "  * Physical map [0.00 Mb -> 46.97 Mb] / Genetic map [0.00 cM -> 46.97 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 163 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 13112 segments / ~3 SNPs per segment / 1267593 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 6971 segments / ~5 SNPs per segment / 154001 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom17.phased.haps]\n",
      "\n",
      "Running time: 34 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:38:15\n",
      "  * LOGfile : [shapeit_30102015_18h38m15s_5365700c-7f4d-41d8-9a49-35ed4ffd3024.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255495\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.map]\n",
      "  * 182 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.18.shapeitmap]\n",
      "  * 182 genetic positions found\n",
      "  * #set=182 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 46.04 Mb] / Genetic map [0.00 cM -> 46.03 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 174 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 14284 segments / ~3 SNPs per segment / 1418480 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 6919 segments / ~6 SNPs per segment / 157488 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom18.phased.haps]\n",
      "\n",
      "Running time: 38 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:38:53\n",
      "  * LOGfile : [shapeit_30102015_18h38m53s_0d885b4b-4f60-40a4-8ee3-bfeeba6acb59.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255533\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.map]\n",
      "  * 206 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.19.shapeitmap]\n",
      "  * 206 genetic positions found\n",
      "  * #set=206 / #interpolated=0\n",
      "  * Physical map [4.71 Mb -> 67.31 Mb] / Genetic map [4.70 cM -> 67.30 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 203 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 16717 segments / ~3 SNPs per segment / 1694669 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 8991 segments / ~5 SNPs per segment / 206353 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom19.phased.haps]\n",
      "\n",
      "Running time: 40 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:39:33\n",
      "  * LOGfile : [shapeit_30102015_18h39m33s_77f8e5f5-23ba-41fc-84aa-c4896acafabc.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255573\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.map]\n",
      "  * 175 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.20.shapeitmap]\n",
      "  * 175 genetic positions found\n",
      "  * #set=175 / #interpolated=0\n",
      "  * Physical map [0.00 Mb -> 85.29 Mb] / Genetic map [0.00 cM -> 85.29 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 167 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 14465 segments / ~2 SNPs per segment / 1387941 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 9240 segments / ~4 SNPs per segment / 215544 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom20.phased.haps]\n",
      "\n",
      "Running time: 49 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:40:22\n",
      "  * LOGfile : [shapeit_30102015_18h40m22s_aebebd68-304b-4696-abc6-8bc329df0660.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255622\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.map]\n",
      "  * 175 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.21.shapeitmap]\n",
      "  * 175 genetic positions found\n",
      "  * #set=175 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 54.72 Mb] / Genetic map [0.00 cM -> 54.71 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 166 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 14047 segments / ~3 SNPs per segment / 1335318 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 7598 segments / ~5 SNPs per segment / 171029 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom21.phased.haps]\n",
      "\n",
      "Running time: 35 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:40:57\n",
      "  * LOGfile : [shapeit_30102015_18h40m57s_6871e6cd-ce84-4a65-a6b9-2f8084976dd9.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255657\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.map]\n",
      "  * 185 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.22.shapeitmap]\n",
      "  * 185 genetic positions found\n",
      "  * #set=185 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 47.24 Mb] / Genetic map [0.00 cM -> 47.23 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 178 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 15274 segments / ~2 SNPs per segment / 1578091 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 7579 segments / ~6 SNPs per segment / 171557 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom22.phased.haps]\n",
      "\n",
      "Running time: 37 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:41:34\n",
      "  * LOGfile : [shapeit_30102015_18h41m34s_fd166f9f-7ee0-498e-a91d-adbfc1856b03.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255694\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.map]\n",
      "  * 196 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.23.shapeitmap]\n",
      "  * 196 genetic positions found\n",
      "  * #set=196 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 54.65 Mb] / Genetic map [0.00 cM -> 54.64 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 185 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 15583 segments / ~3 SNPs per segment / 1528825 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 8186 segments / ~5 SNPs per segment / 186110 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom23.phased.haps]\n",
      "\n",
      "Running time: 38 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:42:12\n",
      "  * LOGfile : [shapeit_30102015_18h42m12s_f02325e0-b7e3-4cad-b046-39aac6059d03.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255732\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.map]\n",
      "  * 156 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.24.shapeitmap]\n",
      "  * 156 genetic positions found\n",
      "  * #set=156 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 59.38 Mb] / Genetic map [0.00 cM -> 59.37 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 150 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 12187 segments / ~3 SNPs per segment / 1145697 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 7094 segments / ~5 SNPs per segment / 163413 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom24.phased.haps]\n",
      "\n",
      "Running time: 36 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:42:48\n",
      "  * LOGfile : [shapeit_30102015_18h42m48s_11cae33c-4a33-40a3-a727-2891b11c5885.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255768\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.map]\n",
      "  * 167 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.25.shapeitmap]\n",
      "  * 167 genetic positions found\n",
      "  * #set=167 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 56.64 Mb] / Genetic map [0.00 cM -> 56.63 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 159 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 14138 segments / ~2 SNPs per segment / 1429167 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 7519 segments / ~5 SNPs per segment / 165414 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom25.phased.haps]\n",
      "\n",
      "Running time: 36 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:43:24\n",
      "  * LOGfile : [shapeit_30102015_18h43m24s_5edadbbd-a0ee-4a80-8ef4-39e38f6b2068.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255804\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.map]\n",
      "  * 219 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.26.shapeitmap]\n",
      "  * 219 genetic positions found\n",
      "  * #set=219 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 76.83 Mb] / Genetic map [0.00 cM -> 76.82 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 214 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 18372 segments / ~2 SNPs per segment / 1815927 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 9655 segments / ~5 SNPs per segment / 208111 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom26.phased.haps]\n",
      "\n",
      "Running time: 50 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:44:14\n",
      "  * LOGfile : [shapeit_30102015_18h44m14s_87a23a37-a917-4e90-97f7-f982bf560ae1.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255854\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.map]\n",
      "  * 153 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.27.shapeitmap]\n",
      "  * 153 genetic positions found\n",
      "  * #set=153 / #interpolated=0\n",
      "  * Physical map [6.24 Mb -> 56.12 Mb] / Genetic map [6.23 cM -> 56.11 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 146 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 12052 segments / ~3 SNPs per segment / 1152694 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 6382 segments / ~5 SNPs per segment / 140258 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom27.phased.haps]\n",
      "\n",
      "Running time: 29 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:44:43\n",
      "  * LOGfile : [shapeit_30102015_18h44m43s_a7c78a8d-dc56-46da-8c70-6df9a61189d4.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255883\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.map]\n",
      "  * 174 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.28.shapeitmap]\n",
      "  * 174 genetic positions found\n",
      "  * #set=174 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 63.10 Mb] / Genetic map [0.00 cM -> 63.09 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 169 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 13693 segments / ~3 SNPs per segment / 1263976 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 7724 segments / ~5 SNPs per segment / 171685 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom28.phased.haps]\n",
      "\n",
      "Running time: 58 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:45:42\n",
      "  * LOGfile : [shapeit_30102015_18h45m42s_5eb1baa0-3414-45cc-aa65-7cfe22dffd13.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255942\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.map]\n",
      "  * 151 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.29.shapeitmap]\n",
      "  * 151 genetic positions found\n",
      "  * #set=151 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 59.79 Mb] / Genetic map [0.00 cM -> 59.78 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 143 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 11805 segments / ~3 SNPs per segment / 1129084 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 6586 segments / ~5 SNPs per segment / 146111 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom29.phased.haps]\n",
      "\n",
      "Running time: 37 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:46:19\n",
      "  * LOGfile : [shapeit_30102015_18h46m19s_6cabf674-567a-425b-8161-5e446ee350c9.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446255979\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.map]\n",
      "  * 137 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.30.shapeitmap]\n",
      "  * 137 genetic positions found\n",
      "  * #set=137 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 50.27 Mb] / Genetic map [0.00 cM -> 50.26 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 133 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 10874 segments / ~3 SNPs per segment / 1045197 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 5213 segments / ~6 SNPs per segment / 111397 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom30.phased.haps]\n",
      "\n",
      "Running time: 32 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:46:51\n",
      "  * LOGfile : [shapeit_30102015_18h46m51s_e14aae08-f5f8-4246-8312-89f290ec4bf9.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446256011\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.map]\n",
      "  * 162 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.31.shapeitmap]\n",
      "  * 162 genetic positions found\n",
      "  * #set=162 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 54.51 Mb] / Genetic map [0.00 cM -> 54.50 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 154 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 13487 segments / ~2 SNPs per segment / 1290927 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 7880 segments / ~5 SNPs per segment / 176072 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom31.phased.haps]\n",
      "\n",
      "Running time: 33 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:47:24\n",
      "  * LOGfile : [shapeit_30102015_18h47m24s_714978b0-f55d-41bf-ac8e-f04a29c0d373.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446256044\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.map]\n",
      "  * 161 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.32.shapeitmap]\n",
      "  * 161 genetic positions found\n",
      "  * #set=161 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 57.99 Mb] / Genetic map [0.00 cM -> 57.98 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 157 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 13175 segments / ~3 SNPs per segment / 1300457 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 6850 segments / ~5 SNPs per segment / 156079 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom32.phased.haps]\n",
      "\n",
      "Running time: 41 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:48:05\n",
      "  * LOGfile : [shapeit_30102015_18h48m05s_9155be10-e043-4eca-8130-4725637ad0a7.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446256085\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.map]\n",
      "  * 125 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.33.shapeitmap]\n",
      "  * 125 genetic positions found\n",
      "  * #set=125 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 36.79 Mb] / Genetic map [0.00 cM -> 36.78 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 123 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 9719 segments / ~3 SNPs per segment / 907625 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 4936 segments / ~6 SNPs per segment / 106990 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom33.phased.haps]\n",
      "\n",
      "Running time: 27 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:48:32\n",
      "  * LOGfile : [shapeit_30102015_18h48m32s_686d6b18-b708-4da3-9ffa-e4261f4b86a2.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446256112\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.map]\n",
      "  * 153 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.34.shapeitmap]\n",
      "  * 153 genetic positions found\n",
      "  * #set=153 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 51.91 Mb] / Genetic map [0.00 cM -> 51.90 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 148 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 13226 segments / ~2 SNPs per segment / 1299141 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 7156 segments / ~5 SNPs per segment / 157229 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom34.phased.haps]\n",
      "\n",
      "Running time: 34 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:49:06\n",
      "  * LOGfile : [shapeit_30102015_18h49m06s_e3758efd-c7d3-4545-a625-e9c7165db194.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446256146\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.map]\n",
      "  * 124 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.35.shapeitmap]\n",
      "  * 124 genetic positions found\n",
      "  * #set=124 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 61.13 Mb] / Genetic map [0.00 cM -> 61.12 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 120 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 10064 segments / ~3 SNPs per segment / 1011203 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 4656 segments / ~6 SNPs per segment / 104966 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom35.phased.haps]\n",
      "\n",
      "Running time: 36 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:49:42\n",
      "  * LOGfile : [shapeit_30102015_18h49m42s_65433baf-23b9-4185-b1e9-2d3863bef2cb.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446256182\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.map]\n",
      "  * 146 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.36.shapeitmap]\n",
      "  * 146 genetic positions found\n",
      "  * #set=146 / #interpolated=0\n",
      "  * Physical map [0.00 Mb -> 57.78 Mb] / Genetic map [0.00 cM -> 57.78 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 140 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 13082 segments / ~2 SNPs per segment / 1299079 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 6704 segments / ~5 SNPs per segment / 148403 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom36.phased.haps]\n",
      "\n",
      "Running time: 51 seconds\n",
      "\n",
      "\u001b[33mS\u001b[0megmented \u001b[33mHAP\u001b[0mlotype \u001b[33mE\u001b[0mstimation & \u001b[33mI\u001b[0mmputation \u001b[33mT\u001b[0mool\n",
      "  * Authors : Olivier Delaneau, Jared O'Connell, Jean-François Zagury, Jonathan Marchini\n",
      "  * Contact : send an email to the OXSTATGEN mail list https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=OXSTATGEN\n",
      "  * Webpage : https://mathgen.stats.ox.ac.uk/shapeit\n",
      "  * Version : v2.r837\n",
      "  * Date    : 30/10/2015 18:50:33\n",
      "  * LOGfile : [shapeit_30102015_18h50m33s_4139180f-2f18-4d5e-a432-182f87b0a5c8.log]\n",
      "\n",
      "MODE \u001b[33m-phase\u001b[0m : PHASING GENOTYPE DATA\n",
      "  * Autosome (chr1 ... chr22)\n",
      "  * Window-based model (SHAPEIT v2)\n",
      "  * MCMC iteration\n",
      "  * duoHMM is used to refine family haplotypes\n",
      "\n",
      "Parameters :\n",
      "  * Seed : 1446256233\n",
      "  * Parallelisation: 1 threads\n",
      "  * Ref allele is NOT aligned on the reference genome\n",
      "  * MCMC: 35 iterations [7 B + 1 runs of 8 P + 20 M]\n",
      "  * Model: 100 states per window [100 H + 0 PM + 0 R + 0 COV ] / Windows of ~5.0 Mb / Ne = 10\n",
      "\n",
      "Reading site list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.map]\n",
      "  * 98 sites included\n",
      "\n",
      "Reading sample list and genotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.ped] where missing-code = [0]\n",
      "  * 246 samples included\n",
      "  * 0 unrelateds / 0 duos / 240 trios in 3 different families\n",
      "\n",
      "Reading genetic map in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.37.shapeitmap]\n",
      "  * 98 genetic positions found\n",
      "  * #set=98 / #interpolated=0\n",
      "  * Physical map [0.01 Mb -> 66.17 Mb] / Genetic map [0.00 cM -> 66.16 cM]\n",
      "\n",
      "Checking Mendel errors...\n",
      "  * Low level of Mendel error in all trios and duos\n",
      "  * 0 SNPs with high Mendel error rate (> 5%)\n",
      "\n",
      "Checking missingness and MAF...\n",
      "  * 240 individuals with high rates of missing data (>5%)\n",
      "  * 96 SNPs with high rates of missing data (>5%)\n",
      "\n",
      "Building graphs [246/246]\n",
      "  * 246 graphs / 7550 segments / ~3 SNPs per segment / 698426 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Sampling haplotypes [246/246]\n",
      "\n",
      "Burn-in iteration [1/7] [246/246]\n",
      "\n",
      "Burn-in iteration [2/7] [246/246]\n",
      "\n",
      "Burn-in iteration [3/7] [246/246]\n",
      "\n",
      "Burn-in iteration [4/7] [246/246]\n",
      "\n",
      "Burn-in iteration [5/7] [246/246]\n",
      "\n",
      "Burn-in iteration [6/7] [246/246]\n",
      "\n",
      "Burn-in iteration [7/7] [246/246]\n",
      "\n",
      "Pruning iteration [1/8] [246/246]\n",
      "\n",
      "Pruning iteration [2/8] [246/246]\n",
      "\n",
      "Pruning iteration [3/8] [246/246]\n",
      "\n",
      "Pruning iteration [4/8] [246/246]\n",
      "\n",
      "Pruning iteration [5/8] [246/246]\n",
      "\n",
      "Pruning iteration [6/8] [246/246]\n",
      "\n",
      "Pruning iteration [7/8] [246/246]\n",
      "\n",
      "Pruning iteration [8/8] [246/246]\n",
      "\n",
      "Pruning graphs [246/246]\n",
      "  * 246 graphs / 4074 segments / ~5 SNPs per segment / 89704 transitions\n",
      "  * 0 haploids / 246 unrelateds / 0 duos / 0 trios\n",
      "  * 492 founder haplotypes\n",
      "\n",
      "Main iteration [1/20] [246/246]\n",
      "\n",
      "Main iteration [2/20] [246/246]\n",
      "\n",
      "Main iteration [3/20] [246/246]\n",
      "\n",
      "Main iteration [4/20] [246/246]\n",
      "\n",
      "Main iteration [5/20] [246/246]\n",
      "\n",
      "Main iteration [6/20] [246/246]\n",
      "\n",
      "Main iteration [7/20] [246/246]\n",
      "\n",
      "Main iteration [8/20] [246/246]\n",
      "\n",
      "Main iteration [9/20] [246/246]\n",
      "\n",
      "Main iteration [10/20] [246/246]\n",
      "\n",
      "Main iteration [11/20] [246/246]\n",
      "\n",
      "Main iteration [12/20] [246/246]\n",
      "\n",
      "Main iteration [13/20] [246/246]\n",
      "\n",
      "Main iteration [14/20] [246/246]\n",
      "\n",
      "Main iteration [15/20] [246/246]\n",
      "\n",
      "Main iteration [16/20] [246/246]\n",
      "\n",
      "Main iteration [17/20] [246/246]\n",
      "\n",
      "Main iteration [18/20] [246/246]\n",
      "\n",
      "Main iteration [19/20] [246/246]\n",
      "\n",
      "Main iteration [20/20] [246/246]\n",
      "\n",
      "Normalising graphs [246/246]\n",
      "\n",
      "Solving haplotypes [246/246]\n",
      "\n",
      "Correcting haplotypes based on pedigree structure...\n",
      "\n",
      "Writing sample list in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.phased.sample]\n",
      "\n",
      "Writing site list and haplotypes in [/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom37.phased.haps]\n",
      "\n",
      "Running time: 28 seconds\n"
     ]
    }
   ],
   "source": [
    "for chrom in range(1, 38):\n",
    "    !'/home/ipseg/Programs/SHAPEIT/shapeit.v2.r837.GLIBCv2.12.Linux.static/bin/shapeit' \\\n",
    "    --input-ped {'/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom{}.ped'.format(chrom)} \\\n",
    "      {'/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom{}.map'.format(chrom)} \\\n",
    "    --duohmm -W 5 \\\n",
    "    --effective-size 10 \\\n",
    "    -- --thread 6 \\\n",
    "    --input-map {'/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.{}.shapeitmap'.format(chrom)} \\\n",
    "    -O {'/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom{}.phased'.format(chrom)}\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Load phase data \n",
    "because of how we code genotypes for LEPmap - we want the sire phase:\n",
    "    - family 8- line 4 \n",
    "    - family 1- line 181\n",
    "    - family 9- line 217\n",
    "    \n",
    "4+2*(x-2) = (8,9) (362, 363), [434, 435]\n",
    "\n",
    "    \n",
    "this corresponds to columns:\n",
    "    - [7,8]\n",
    "    - [361, 362]\n",
    "    - [433, 434]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>361</th>\n",
       "      <th>362</th>\n",
       "      <th>433</th>\n",
       "      <th>434</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>256</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>257</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>258</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>259</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>260</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>261</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>262</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>263</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>264</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>265</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>266</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>267</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>268</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>269</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>270</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>271</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>272</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>273</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>274</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>275</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>276</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>277</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>278</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>279</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>280</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>281</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>282</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>283</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>284</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>285</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>286 rows × 6 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     7    8    361  362  433  434\n",
       "0      1    0    1    0    1    0\n",
       "1      0    1    1    0    0    1\n",
       "2      1    0    0    1    1    0\n",
       "3      1    0    0    1    1    0\n",
       "4      1    0    0    1    1    0\n",
       "5      0    1    0    1    0    1\n",
       "6      0    1    0    1    0    1\n",
       "7      1    0    0    1    1    0\n",
       "8      1    0    0    1    1    0\n",
       "9      1    0    0    1    1    0\n",
       "10     1    0    0    1    1    0\n",
       "11     1    0    0    1    1    0\n",
       "12     1    0    0    1    1    0\n",
       "13     1    0    0    1    0    1\n",
       "14     1    0    0    1    0    1\n",
       "15     1    0    0    1    0    1\n",
       "16     1    0    0    1    0    1\n",
       "17     0    1    0    1    1    0\n",
       "18     0    1    0    1    1    0\n",
       "19     0    1    1    0    1    0\n",
       "20     1    0    1    0    1    0\n",
       "21     1    0    1    0    1    0\n",
       "22     1    0    1    0    1    0\n",
       "23     0    1    1    0    0    1\n",
       "24     1    0    1    0    1    0\n",
       "25     0    1    1    0    1    0\n",
       "26     1    0    1    0    1    0\n",
       "27     0    1    0    1    1    0\n",
       "28     1    0    1    0    0    1\n",
       "29     0    1    0    1    1    0\n",
       "..   ...  ...  ...  ...  ...  ...\n",
       "256    1    0    1    0    0    1\n",
       "257    0    1    0    1    0    1\n",
       "258    0    1    0    1    0    1\n",
       "259    1    0    1    0    0    1\n",
       "260    0    1    0    1    1    0\n",
       "261    1    0    1    0    0    1\n",
       "262    1    0    1    0    0    1\n",
       "263    0    1    0    1    0    1\n",
       "264    0    1    0    1    0    1\n",
       "265    0    1    0    1    0    1\n",
       "266    1    0    0    1    0    1\n",
       "267    0    1    0    1    1    0\n",
       "268    0    1    1    0    0    1\n",
       "269    1    0    1    0    0    1\n",
       "270    1    0    0    1    0    1\n",
       "271    1    0    1    0    0    1\n",
       "272    1    0    1    0    0    1\n",
       "273    1    0    1    0    0    1\n",
       "274    1    0    1    0    0    1\n",
       "275    0    1    1    0    0    1\n",
       "276    0    1    1    0    0    1\n",
       "277    0    1    1    0    0    1\n",
       "278    1    0    0    1    1    0\n",
       "279    0    1    0    1    1    0\n",
       "280    0    1    1    0    1    0\n",
       "281    1    0    0    1    1    0\n",
       "282    1    0    0    1    1    0\n",
       "283    0    1    0    1    0    1\n",
       "284    1    0    0    1    0    1\n",
       "285    1    0    0    1    0    1\n",
       "\n",
       "[286 rows x 6 columns]"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.read_clipboardphased = pd.read_csv('/home/ipseg/Desktop/waples/chum_populations/linkage_map/centromere_placement/shapeit/collapsed_loci.chrom9.phased.haps', \n",
    "           sep = ' ', header = None)\n",
    "\n",
    "fam8 = chr9_phased.iloc[:,[7,8]]\n",
    "fam1 = chr9_phased.iloc[:,[361, 362]]\n",
    "fam9 = chr9_phased.iloc[:,[433, 434]]\n",
    "\n",
    "pd.merge(pd.merge(fam8, fam1, right_index=True, left_index=True), fam9, right_index=True, left_index=True)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>362</th>\n",
       "      <th>363</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>256</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>257</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>258</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>259</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>260</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>261</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>262</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>263</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>264</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>265</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>266</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>267</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>268</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>269</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>270</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>271</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>272</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>273</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>274</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>275</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>276</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>277</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>278</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>279</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>280</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>281</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>282</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>283</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>284</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>285</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>286 rows × 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     362  363\n",
       "0      0    0\n",
       "1      0    0\n",
       "2      1    1\n",
       "3      1    0\n",
       "4      1    0\n",
       "5      1    0\n",
       "6      1    0\n",
       "7      1    0\n",
       "8      1    0\n",
       "9      1    0\n",
       "10     1    0\n",
       "11     1    0\n",
       "12     1    0\n",
       "13     1    0\n",
       "14     1    0\n",
       "15     1    0\n",
       "16     1    0\n",
       "17     1    0\n",
       "18     1    0\n",
       "19     0    1\n",
       "20     0    0\n",
       "21     0    0\n",
       "22     0    0\n",
       "23     0    0\n",
       "24     0    0\n",
       "25     0    0\n",
       "26     0    1\n",
       "27     1    0\n",
       "28     0    1\n",
       "29     1    0\n",
       "..   ...  ...\n",
       "256    0    0\n",
       "257    1    0\n",
       "258    1    0\n",
       "259    0    0\n",
       "260    1    0\n",
       "261    0    0\n",
       "262    0    0\n",
       "263    1    0\n",
       "264    1    0\n",
       "265    1    0\n",
       "266    1    0\n",
       "267    1    0\n",
       "268    0    0\n",
       "269    0    0\n",
       "270    1    0\n",
       "271    0    0\n",
       "272    0    0\n",
       "273    0    0\n",
       "274    0    0\n",
       "275    0    0\n",
       "276    0    0\n",
       "277    0    0\n",
       "278    1    0\n",
       "279    1    0\n",
       "280    0    0\n",
       "281    1    0\n",
       "282    1    0\n",
       "283    1    0\n",
       "284    1    0\n",
       "285    1    0\n",
       "\n",
       "[286 rows x 2 columns]"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "chr9_phased.iloc[:,[362,363]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>362</th>\n",
       "      <th>363</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>256</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>257</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>258</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>259</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>260</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>261</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>262</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>263</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>264</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>265</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>266</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>267</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>268</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>269</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>270</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>271</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>272</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>273</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>274</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>275</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>276</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>277</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>278</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>279</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>280</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>281</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>282</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>283</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>284</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>285</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>286 rows × 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     8    9    362  363\n",
       "0      0    0    0    0\n",
       "1      1    0    0    0\n",
       "2      0    0    1    1\n",
       "3      0    0    1    0\n",
       "4      0    0    1    0\n",
       "5      1    0    1    0\n",
       "6      1    0    1    0\n",
       "7      0    0    1    0\n",
       "8      0    0    1    0\n",
       "9      0    0    1    0\n",
       "10     0    0    1    0\n",
       "11     0    0    1    0\n",
       "12     0    0    1    0\n",
       "13     0    0    1    0\n",
       "14     0    0    1    0\n",
       "15     0    0    1    0\n",
       "16     0    0    1    0\n",
       "17     1    0    1    0\n",
       "18     1    0    1    0\n",
       "19     1    0    0    1\n",
       "20     0    1    0    0\n",
       "21     0    1    0    0\n",
       "22     0    1    0    0\n",
       "23     1    0    0    0\n",
       "24     0    1    0    0\n",
       "25     1    0    0    0\n",
       "26     0    0    0    1\n",
       "27     1    0    1    0\n",
       "28     0    1    0    1\n",
       "29     1    0    1    0\n",
       "..   ...  ...  ...  ...\n",
       "256    0    0    0    0\n",
       "257    1    0    1    0\n",
       "258    1    0    1    0\n",
       "259    0    0    0    0\n",
       "260    1    0    1    0\n",
       "261    0    0    0    0\n",
       "262    0    0    0    0\n",
       "263    1    1    1    0\n",
       "264    1    0    1    0\n",
       "265    1    1    1    0\n",
       "266    0    0    1    0\n",
       "267    1    1    1    0\n",
       "268    1    0    0    0\n",
       "269    0    0    0    0\n",
       "270    0    0    1    0\n",
       "271    0    0    0    0\n",
       "272    0    0    0    0\n",
       "273    0    0    0    0\n",
       "274    0    0    0    0\n",
       "275    1    1    0    0\n",
       "276    1    1    0    0\n",
       "277    1    0    0    0\n",
       "278    0    0    1    0\n",
       "279    1    1    1    0\n",
       "280    1    1    0    0\n",
       "281    0    0    1    0\n",
       "282    0    0    1    0\n",
       "283    1    0    1    0\n",
       "284    0    0    1    0\n",
       "285    0    0    1    0\n",
       "\n",
       "[286 rows x 4 columns]"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.merge(chr9_phased.iloc[:,[8,9]], chr9_phased.iloc[:,[362,363]], left_index=True, right_index=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
